# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# dcantrel <dcantrell@redhat.com>, 2011.
# Dimitris Glezos <glezos@indifex.com>, 2011.
# Hajime Taira <htaira@redhat.com>, 2010-2012.
# Hirofumi Saito <hi_saito@yk.rim.or.jp>, 2004-2006.
# Hyu_gabaru Ryu_ichi <hyu_gabaru@yahoo.co.jp>, 2007,2009.
# James Hashida <khashida@redhat.com>, 2002-2003.
# Kiyoto Hashida <khashida@redhat.com>, 2007-2010,2012.
# Kiyoto James Hashida <khashida@redhat.com>, 2002-2003,2006-2010.
# Makoto Mizukami <mmk9060@aol.com>, 2008.
# Naoki Shigematsu <sigematu@linuxml.net>, 2004.
#   <noriko@fedoraproject.org>, 2012.
# Noriko Mizumoto <noriko@fedoraproject.org>, 2010.
# Noriko Mizumoto <noriko@redhat.com>, 2003-2004,2006,2008-2009.
# SATO Satoru <ssato@redhat.com>, 2001.
# Tadashi Jokagi <fedora@elf.no-ip.org>, 2004-2005.
# Tomoyuki KATO <tomo@dream.daynight.jp>, 2011-2012.
#   <www.carrotsoft@gmail.com>, 2012.
# Yoshinari Takaoka <mumumu@mx.edit.ne.jp>, 2004.
msgid ""
msgstr ""
"Project-Id-Version: Anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2014-12-02 09:06+0100\n"
"PO-Revision-Date: 2013-01-04 23:55+0000\n"
"Last-Translator: Brian Lane <bcl@redhat.com>\n"
"Language-Team: Japanese <trans-ja@lists.fedoraproject.org>\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: pyanaconda/bootloader.py:247
msgid "/boot filesystem"
msgstr "/boot ファイルシステム"

#: pyanaconda/bootloader.py:404
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following raid "
"levels: %(raid_level)s."
msgstr "'%s' を含む RAID セットは、次の RAID レベルのいずれかが必要です: %s"

#: pyanaconda/bootloader.py:412
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following metadata "
"versions: %(metadata_versions)s."
msgstr ""
"'%s' を含む RAID セットは、次のメタデータのバージョンのいずれかが必要です: %s"

#: pyanaconda/bootloader.py:420
#, fuzzy, python-format
msgid ""
"RAID sets that contain '%(desc)s' must have one of the following device "
"types: %(types)s."
msgstr "'%s' を含む RAID セットは、次のデバイスタイプのいずれかが必要です: %s"

#: pyanaconda/bootloader.py:436
#, fuzzy, python-format
msgid "%(name)s must have one of the following disklabel types: %(types)s."
msgstr "%s は次のディスクラベルのタイプのいずれかが必要です: %s"

#: pyanaconda/bootloader.py:448
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s."
msgstr "%s は %s タイプにすることはできません。"

#: pyanaconda/bootloader.py:454
#, fuzzy, python-format
msgid "%(desc)s must be mounted on one of %(mountpoints)s."
msgstr "%s を %s の１つにマウントする必要があります。"

#: pyanaconda/bootloader.py:466
#, fuzzy, python-format
msgid "%(desc)s must be between %(min)d and %(max)d MB in size"
msgstr "%s のサイズは %d MB から %d MB の間でなければなりません。"

#: pyanaconda/bootloader.py:472
#, fuzzy, python-format
msgid "%(desc)s must not be smaller than %(min)dMB."
msgstr "%s のサイズは %d MB 以下でなければなりません。"

#: pyanaconda/bootloader.py:481
#, fuzzy, python-format
msgid "%(desc)s must not be larger than %(max)dMB."
msgstr "%s のサイズは %d MB 以上でなければなりません。"

#: pyanaconda/bootloader.py:499
#, fuzzy, python-format
msgid "%(desc)s must be within the first %(max_end)s of the disk."
msgstr "%s はディスクの先頭から %dMB 以内の位置に配置しなければなりません。"

#: pyanaconda/bootloader.py:509
#, python-format
msgid "%s must be on a primary partition."
msgstr "%s はプライマリパーティション上になければなりません。"

#: pyanaconda/bootloader.py:645 pyanaconda/bootloader.py:745
#, python-format
msgid "%s cannot be on an encrypted block device."
msgstr "%s は暗号化ブロックデバイス上に置くことができません。"

#: pyanaconda/bootloader.py:700
#, python-format
msgid "%s cannot be on an iSCSI disk on s390(x)"
msgstr ""

#: pyanaconda/bootloader.py:704
#, fuzzy, python-format
msgid "%(desc)s cannot be of type %(type)s"
msgstr "%s は %s タイプにすることはできません。"

#: pyanaconda/bootloader.py:1348
#, python-format
msgid ""
"bootloader stage2 device %(stage2dev)s is on a multi-disk array, but "
"bootloader stage1 device %(stage1dev)s is not. A drive failure in "
"%(stage2dev)s could render the system unbootable."
msgstr ""

#: pyanaconda/bootloader.py:1354
#, python-format
msgid ""
"bootloader stage2 device %(stage2dev)s is on a multi-disk array, but "
"bootloader stage1 device %(stage1dev)s is not part of this array. The stage1 "
"bootloader will only be installed to a single drive."
msgstr ""

#: pyanaconda/bootloader.py:1645
#, fuzzy, python-format
msgid ""
"%(deviceName)s may not have enough space for grub2 to embed core.img when "
"using the %(fsType)s file system on %(deviceType)s"
msgstr ""
"%3s 上の %2s ファイルシステムを使用している場合、 grub2によって組み込まれる "
"core.img 用の領域が %1s には不足している可能性があります。"

#: pyanaconda/constants.py:72
msgid "Start VNC"
msgstr "VNC を起動する"

#: pyanaconda/constants.py:73
msgid "Use text mode"
msgstr "テキストモードを使用する"

#: pyanaconda/constants.py:137
#, fuzzy
msgid "The password is empty."
msgstr "パスワードは設定されています"

#: pyanaconda/constants.py:138
#, fuzzy
msgid "The passwords do not match."
msgstr "パスワードが一致しません！"

#: pyanaconda/constants.py:139
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/constants.py:140
msgid ""
"The password you have provided is weak. You will have to press Done twice to "
"confirm it."
msgstr ""

#: pyanaconda/constants.py:141
#, python-format
msgid ""
"The password you have provided is weak: %s. You will have to press Done "
"twice to confirm it."
msgstr ""

#: pyanaconda/constants.py:142
#, fuzzy
msgid "You have provided a weak password. Press Done again to use anyway."
msgstr "脆弱なパスワードです: %s　それでも使用する場合は完了を押します。"

#: pyanaconda/constants.py:143
#, fuzzy, python-format
msgid "You have provided a weak password: %s. Press Done again to use anyway."
msgstr "脆弱なパスワードです: %s　それでも使用する場合は完了を押します。"

#: pyanaconda/constants.py:144
msgid ""
"The password you have provided contains non-ASCII characters. You may not be "
"able to switch between keyboard layouts to login. Press Done to continue."
msgstr ""

#: pyanaconda/constants.py:146
msgid "Empty"
msgstr ""

#: pyanaconda/constants.py:146 pyanaconda/ui/gui/spokes/lib/passphrase.py:118
#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:177
msgid "Weak"
msgstr "脆弱"

#: pyanaconda/constants.py:146 pyanaconda/ui/gui/spokes/lib/passphrase.py:121
msgid "Fair"
msgstr "普通"

#: pyanaconda/constants.py:146 pyanaconda/ui/gui/spokes/lib/passphrase.py:124
msgid "Good"
msgstr "良好"

#: pyanaconda/constants.py:146 pyanaconda/ui/gui/spokes/lib/passphrase.py:127
msgid "Strong"
msgstr "強力"

#: pyanaconda/constants_text.py:40 pyanaconda/rescue.py:333
#: pyanaconda/rescue.py:365 pyanaconda/rescue.py:378 pyanaconda/rescue.py:446
#: pyanaconda/rescue.py:462 pyanaconda/rescue.py:473 anaconda:644
msgid "OK"
msgstr "OK"

#: pyanaconda/constants_text.py:44 pyanaconda/text.py:52 pyanaconda/text.py:53
#, fuzzy
msgid "Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/constants_text.py:48
#, fuzzy
msgid "Yes"
msgstr "はい(_Y)"

#: pyanaconda/constants_text.py:52
#, fuzzy
msgid "No"
msgstr "いいえ(_N)"

#: pyanaconda/errors.py:104
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"The installation will now terminate."
msgstr ""
"インストールリポジトリーを設定中に次のエラーが発生しました。\n"
"\n"
"%(e)s\n"
"\n"
"インストール作業を継続できません。"

#: pyanaconda/errors.py:110
#, python-format
msgid "An error occurred while resizing the device %s."
msgstr ""

#: pyanaconda/errors.py:119
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems.  Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/errors.py:126
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: pyanaconda/errors.py:137
msgid ""
"There is an entry in your /etc/fstab file that contains an invalid or "
"incorrect filesystem type:\n"
"\n"
msgstr ""

#: pyanaconda/errors.py:143
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error. Would you like to continue "
"using this image?"
msgstr ""

#: pyanaconda/errors.py:157
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Should I try again to locate the image?"
msgstr ""

#: pyanaconda/errors.py:167
#, python-format
msgid "An error occurred mounting the source device %s. Retry?"
msgstr ""

#: pyanaconda/errors.py:175
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/errors.py:183
#, python-format
msgid ""
"The group '%s' is required for this installation. This group does not exist. "
"This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:189
#, python-format
msgid ""
"You have specified that the group '%s' should be installed.  This group does "
"not exist.  Would you like to ignore this group and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:194
#, python-format
msgid ""
"You have specified that the group '%s' should be excluded from "
"installation.  This group does not exist.  Would you like to ignore this "
"group and continue with installation?"
msgstr ""

#: pyanaconda/errors.py:206
#, python-format
msgid ""
"The package '%s' is required for this installation. This package does not "
"exist. This is a fatal error and installation will be aborted."
msgstr ""

#: pyanaconda/errors.py:213
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist.  Would you like to ignore this package and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:224
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  This is "
"a fatal error and installation will be aborted.  The details of this error "
"are:\n"
"\n"
"%(details)s"
msgstr ""

#: pyanaconda/errors.py:232
#, fuzzy
msgid ""
"The following error occurred while installing.  This is a fatal error and "
"installation will be aborted."
msgstr ""
"インストールリポジトリーを設定中に次のエラーが発生しました。\n"
"\n"
"%(e)s\n"
"\n"
"インストール作業を継続できません。"

#: pyanaconda/errors.py:240
#, fuzzy
msgid ""
"The following software marked for installation has errors.  This is likely "
"caused by an error with\n"
"your installation source."
msgstr ""
"インストール用にマークしてある以下のソフトウェアにエラーがあります。これはイ"
"ンストールソースが持つエラーが原因であるようです。これらのパッケージをインス"
"トールから除外するか、インストールソースを変えるか、またはインストーラーを終"
"了することができます。"

#: pyanaconda/errors.py:248
msgid ""
"The following error occurred while installing the bootloader. The system "
"will not be bootable. Would you like to ignore this and continue with "
"installation?"
msgstr ""

#: pyanaconda/errors.py:261
#, fuzzy, python-format
msgid ""
"Installation was stopped due to an error installing the boot loader. The "
"exact error message is:\n"
"\n"
"%s\n"
"\n"
"The installer will now terminate."
msgstr ""
"インストール作業はハードウェアに問題があると思われる症状が発生したため停止さ"
"れました。正確なエラーメッセージは次のとおりです：\n"
"\n"
"%s\n"
"\n"
" インストーラーを終了します。"

#: pyanaconda/exception.py:96
#, python-format
msgid ""
"The installation was stopped due to what seems to be a problem with your "
"hardware. The exact error message is:\n"
"\n"
"%s.\n"
"\n"
" The installer will now terminate."
msgstr ""
"インストール作業はハードウェアに問題があると思われる症状が発生したため停止さ"
"れました。正確なエラーメッセージは次のとおりです：\n"
"\n"
"%s\n"
"\n"
" インストーラーを終了します。"

#: pyanaconda/exception.py:100
#, fuzzy
msgid "Hardware error occured"
msgstr "ハードウェアエラーが発生しました。"

#: pyanaconda/exception.py:134
#, fuzzy, python-format
msgid ""
"\n"
"The installation was stopped due to incomplete spokes detected while running "
"in non-interactive cmdline mode. Since there can not be any questions in "
"cmdline mode, edit your kickstart file and retry installation.\n"
"The exact error message is: \n"
"\n"
"%s.\n"
"\n"
"The installer will now terminate."
msgstr ""
"インストール作業はハードウェアに問題があると思われる症状が発生したため停止さ"
"れました。正確なエラーメッセージは次のとおりです：\n"
"\n"
"%s\n"
"\n"
" インストーラーを終了します。"

#: pyanaconda/install.py:77
msgid "Configuring installed system"
msgstr "インストール済みのシステムを構成"

#: pyanaconda/install.py:90
msgid "Writing network configuration"
msgstr "ネットワーク構成を書き込み中"

#: pyanaconda/install.py:94
msgid "Creating users"
msgstr "ユーザーを作成中"

#: pyanaconda/install.py:101
#, fuzzy
msgid "Configuring addons"
msgstr "インストール済みのシステムを構成"

#: pyanaconda/install.py:104
msgid "Generating initramfs"
msgstr ""

#: pyanaconda/install.py:108
#, python-format
msgid "Joining realm: %s"
msgstr ""

#: pyanaconda/install.py:111
#, fuzzy
msgid "Running post-installation scripts"
msgstr "インストール前スクリプトを実行中"

#: pyanaconda/install.py:165
#, python-format
msgid "Waiting for %s threads to finish"
msgstr ""

#: pyanaconda/install.py:171
#, fuzzy
msgid "Setting up the installation environment"
msgstr "インストールソースをセットアップ中..."

#: pyanaconda/install.py:203
msgid "Discovering realm to join"
msgstr ""

#: pyanaconda/install.py:254
msgid "Installing bootloader"
msgstr "ブートローダーをインストール中"

#: pyanaconda/install.py:257 pyanaconda/packaging/livepayload.py:170
#: pyanaconda/packaging/yumpayload.py:1390
#: pyanaconda/packaging/dnfpayload.py:539
#, fuzzy
msgid "Performing post-installation setup tasks"
msgstr "インストール後設定タスクを実行中"

#: pyanaconda/iutil.py:541
msgid "No host url"
msgstr ""

#: pyanaconda/iutil.py:562
msgid "malformed URL, cannot parse it."
msgstr ""

#: pyanaconda/iutil.py:578
msgid "URL has no host component"
msgstr ""

#: pyanaconda/kickstart.py:157
#, python-format
msgid "Escrow certificate %s requires the network."
msgstr "エスクロー証明書 %s にはネットワークが必要です。"

#: pyanaconda/kickstart.py:165
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"escrow 証明書のダウンロード中に次のエラーに 遭遇しました:\n"
"\n"
"%s"

#: pyanaconda/kickstart.py:274
#, fuzzy, python-format
msgid "%s is missing. Cannot setup authentication."
msgstr "%s が足りません。 ファイアーウォールの設定ができません。"

#: pyanaconda/kickstart.py:299
#, python-format
msgid "autopart fstype of %s is invalid."
msgstr ""

#: pyanaconda/kickstart.py:316
#, python-format
msgid "Settings default fstype to %s failed."
msgstr ""

#: pyanaconda/kickstart.py:347
msgid "GRUB2 does not support installation to a partition."
msgstr ""

#: pyanaconda/kickstart.py:402
#, python-format
msgid "More than one match found for given boot drive \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:405
#, python-format
msgid "Requested boot drive \"%s\" doesn't exist or cannot be used."
msgstr ""

#: pyanaconda/kickstart.py:447
#, python-format
msgid ""
"BTRFS partition \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"btrfs\"."
msgstr ""

#: pyanaconda/kickstart.py:452
#, python-format
msgid "Tried to use undefined partition \"%s\" in BTRFS volume specification."
msgstr ""

#: pyanaconda/kickstart.py:465
msgid ""
"BTRFS volume defined without any member devices.  Either specify member "
"devices or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:474 pyanaconda/kickstart.py:852
#: pyanaconda/kickstart.py:1393
#, python-format
msgid "The mount point \"%s\" is not valid.  It must start with a /."
msgstr ""

#: pyanaconda/kickstart.py:489
#, python-format
msgid "BTRFS volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:585
#, python-format
msgid ""
"Disklabel \"%s\" given in clearpart command is not supported on this "
"platform."
msgstr ""

#: pyanaconda/kickstart.py:597
#, python-format
msgid "Disk \"%s\" given in clearpart command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:610
#, python-format
msgid "Device \"%s\" given in clearpart device list does not exist."
msgstr ""

#: pyanaconda/kickstart.py:637
#, python-format
msgid "NIC \"%s\" given in fcoe command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:688
#, python-format
msgid "%s is missing. Cannot setup firewall."
msgstr "%s が足りません。 ファイアーウォールの設定ができません。"

#: pyanaconda/kickstart.py:736 pyanaconda/kickstart.py:747
#, python-format
msgid "Disk \"%s\" given in ignoredisk command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:760
#, python-format
msgid ""
"Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target "
"is not up."
msgstr ""

#: pyanaconda/kickstart.py:770
msgid ""
"iscsi --iface must be specified (binding used) either for all targets or for "
"none"
msgstr ""

#: pyanaconda/kickstart.py:858
#, python-format
msgid ""
"No volume group exists with the name \"%s\".  Specify volume groups before "
"logical volumes."
msgstr ""

#: pyanaconda/kickstart.py:865
#, python-format
msgid ""
"No thin pool exists with the name \"%s\". Specify thin pools before thin "
"volumes."
msgstr ""

#: pyanaconda/kickstart.py:873
msgid "logvol --noformat must also use the --name= option."
msgstr ""

#: pyanaconda/kickstart.py:878 pyanaconda/kickstart.py:946
#, python-format
msgid "Logical volume \"%s\" given in logvol command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:889 pyanaconda/kickstart.py:898
#: pyanaconda/kickstart.py:956 pyanaconda/kickstart.py:1196
#: pyanaconda/kickstart.py:1205 pyanaconda/kickstart.py:1286
#, python-format
msgid "Target size \"%(size)s\" for device \"%(device)s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:912
#, python-format
msgid ""
"Logical volume name \"%(logvol)s\" is already in use in volume group "
"\"%(volgroup)s\"."
msgstr ""

#: pyanaconda/kickstart.py:919
#, python-format
msgid ""
"No size given for logical volume \"%s\".  Use one of --useexisting, --size, "
"or --percent."
msgstr ""

#: pyanaconda/kickstart.py:922
#, python-format
msgid ""
"Logical volume size \"%(logvolSize)s\" must be larger than the volume group "
"extent size of \"%(extentSize)s\"."
msgstr ""

#: pyanaconda/kickstart.py:926
msgid "Percentage must be between 0 and 100."
msgstr ""

#: pyanaconda/kickstart.py:936 pyanaconda/kickstart.py:1223
#: pyanaconda/kickstart.py:1443
#, fuzzy, python-format
msgid "The \"%s\" filesystem type is not supported."
msgstr "このファイルシステムはラベルをサポートしません。"

#: pyanaconda/kickstart.py:1064 pyanaconda/kickstart.py:1068
#, python-format
msgid "The %s kickstart command is not currently supported."
msgstr ""

#: pyanaconda/kickstart.py:1093
#, python-format
msgid "No disk found for specified BIOS disk \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1128
#, python-format
msgid "RAID partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1139 pyanaconda/kickstart.py:1368
#, python-format
msgid "PV partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1150 pyanaconda/kickstart.py:1378
#, python-format
msgid "BTRFS partition \"%s\" is defined multiple times."
msgstr ""

#: pyanaconda/kickstart.py:1173
#, python-format
msgid "The size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1180
msgid "part --noformat must also use the --onpart option."
msgstr ""

#: pyanaconda/kickstart.py:1185 pyanaconda/kickstart.py:1277
#, python-format
msgid "Partition \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1238 pyanaconda/kickstart.py:1252
#, python-format
msgid "Disk \"%s\" given in part command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1241
#, python-format
msgid "Cannot install to unpartitionable device \"%s\"."
msgstr ""

#: pyanaconda/kickstart.py:1248
#, python-format
msgid "Disk \"%s\" in part command is not partitioned."
msgstr ""

#: pyanaconda/kickstart.py:1261
#, python-format
msgid "The maximum size \"%s\" is invalid."
msgstr ""

#: pyanaconda/kickstart.py:1400
msgid "raid --noformat must also use the --device option."
msgstr ""

#: pyanaconda/kickstart.py:1405
#, python-format
msgid "RAID device  \"%s\" given in raid command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1426
#, python-format
msgid ""
"RAID device \"%(device)s\" has a format of \"%(format)s\", but should have a "
"format of \"mdmember\"."
msgstr ""

#: pyanaconda/kickstart.py:1431
#, python-format
msgid "Tried to use undefined partition \"%s\" in RAID specification."
msgstr ""

#: pyanaconda/kickstart.py:1459
#, python-format
msgid "RAID volume \"%s\" specified with --useexisting does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1466
#, fuzzy, python-format
msgid "The RAID volume name \"%s\" is already in use."
msgstr "指定された名前 %s はすでに存在します。"

#: pyanaconda/kickstart.py:1676
#, python-format
msgid ""
"Physical volume \"%(device)s\" has a format of \"%(format)s\", but should "
"have a format of \"lvmpv\"."
msgstr ""

#: pyanaconda/kickstart.py:1681
#, python-format
msgid "Tried to use undefined partition \"%s\" in Volume Group specification"
msgstr ""

#: pyanaconda/kickstart.py:1687
#, python-format
msgid ""
"Volume group \"%s\" defined without any physical volumes.  Either specify "
"physical volumes or use --useexisting."
msgstr ""

#: pyanaconda/kickstart.py:1697
#, python-format
msgid ""
"Volume group given physical extent size of \"%(extentSize)s\", but must be "
"one of:\n"
"%(validExtentSizes)s."
msgstr ""

#: pyanaconda/kickstart.py:1704
msgid ""
"volgroup --noformat and volgroup --useexisting must also use the --name= "
"option."
msgstr ""

#: pyanaconda/kickstart.py:1709
#, python-format
msgid "Volume group \"%s\" given in volgroup command does not exist."
msgstr ""

#: pyanaconda/kickstart.py:1712
#, fuzzy, python-format
msgid "The volume group name \"%s\" is already in use."
msgstr "ボリュームグループ名 %s はすでに使われています。変更は保存されません。"

#: pyanaconda/kickstart.py:1765
msgid ""
"The upgrade kickstart command is no longer supported. Upgrade functionality "
"is provided through fedup."
msgstr ""

#: pyanaconda/kickstart.py:1980
msgid "Running pre-installation scripts"
msgstr "インストール前スクリプトを実行中"

#: pyanaconda/network.py:103
msgid "Hostname cannot be None or an empty string."
msgstr "ホスト名なし、または空白の文字は設定できません。"

#: pyanaconda/network.py:106
msgid "Hostname must be 255 or fewer characters in length."
msgstr "ホスト名は 255 文字かそれ以下の長さにする必要があります。"

#: pyanaconda/network.py:109
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.', "
"parts between periods must contain something and cannot start or end with "
"'-'."
msgstr ""
" ホスト名には半角英数 'a-z'、'A-Z'、'0-9'、および記号 '-'、'.' を使用すること"
"ができます。ただし、ピリオドは文字列の途中に使用しなければいけません。また、"
"ホスト名の先頭と終わりには '-' は使用できません。"

#: pyanaconda/network.py:1265 pyanaconda/ui/gui/spokes/custom.py:518
#: pyanaconda/ui/gui/spokes/network.py:297
#: pyanaconda/ui/gui/spokes/lib/accordion.py:61
msgid "Unknown"
msgstr "不明"

#: pyanaconda/network.py:1269
msgid "Connecting..."
msgstr "接続中..."

#: pyanaconda/network.py:1271
msgid "Disconnecting..."
msgstr "切断中..."

#: pyanaconda/network.py:1291
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) connected"
msgstr "有線 (%s) は接続されています"

#: pyanaconda/network.py:1294
#, fuzzy, python-format
msgid "Wireless connected to %(access_point)s"
msgstr "%s にワイヤレスで接続しています"

#: pyanaconda/network.py:1297
#, python-format
msgid "Bond %(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1301
#, python-format
msgid "Team%(interface_name)s (%(list_of_slaves)s) connected"
msgstr ""

#: pyanaconda/network.py:1307
#, python-format
msgid "Vlan %(interface_name)s (%(parent_device)s, ID %(vlanid)s) connected"
msgstr ""

#: pyanaconda/network.py:1322
#, python-format
msgid "Connected: %(list_of_interface_names)s"
msgstr ""

#: pyanaconda/network.py:1325
msgid "Not connected"
msgstr "接続していません"

#: pyanaconda/network.py:1328 pyanaconda/ui/gui/spokes/network.glade:2099
msgid "No network devices available"
msgstr "ネットワークデバイスは利用できません"

#: pyanaconda/product.py:53
#, python-format
msgid "%(productName)s %(productVersion)s INSTALLATION"
msgstr "%(productName)s %(productVersion)s インストール"

#: pyanaconda/product.py:57
#, fuzzy, python-format
msgid "New %(name)s %(version)s Installation"
msgstr "新しい %s %s のインストール"

#: pyanaconda/rescue.py:183 pyanaconda/rescue.py:350 pyanaconda/rescue.py:439
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr ""
"メンテナンスが終わったら、システムをアンマウントするために %s を実行してくだ"
"さい。"

#: pyanaconda/rescue.py:186
msgid "When finished please exit from the shell and your system will reboot."
msgstr "完了したらシェルを終了してください。システムが再起動します。"

#: pyanaconda/rescue.py:200
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr "実行するための /bin/sh が見付かりません。シェルを 開始していません。"

#: pyanaconda/rescue.py:288 pyanaconda/rescue.py:361 pyanaconda/rescue.py:371
#: pyanaconda/rescue.py:442 pyanaconda/rescue.py:473
msgid "Rescue"
msgstr "レスキュー"

#: pyanaconda/rescue.py:289
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  \n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"レスキュー環境はここで、ユーザーの Linux インストールを見つけて、それをディレ"
"クトリ %s にマウントする試みをします。その後に、必要な変更をシステムに加える"
"ことができます。このステップを進めたい場合は、「続ける」を選択します。また、"
"「読み込み専用」を選択すると、読み込み/書き込みの代わりに読み込み専用でファイ"
"ルシステムをマウントする選択ができます。\n"
"\n"
"なんらかの理由でこのプロセスが失敗した場合は、「スキップ」を選択すると、この"
"ステップは除外されて直接、コマンドシェルに進みます。\n"

#: pyanaconda/rescue.py:299
msgid "Continue"
msgstr "続行"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:305
msgid "Read-Only"
msgstr "読み取り専用"

#: pyanaconda/rescue.py:299 pyanaconda/rescue.py:301
msgid "Skip"
msgstr "スキップする"

#: pyanaconda/rescue.py:330
msgid "System to Rescue"
msgstr "レスキュー対象システム"

#: pyanaconda/rescue.py:331
msgid "Which device holds the root partition of your installation?"
msgstr "どのデバイスにインストールの root パーティションを置きますか？"

#: pyanaconda/rescue.py:333 pyanaconda/rescue.py:337
msgid "Exit"
msgstr "終了"

#: pyanaconda/rescue.py:347 pyanaconda/rescue.py:436
msgid "The system will reboot automatically when you exit from the shell."
msgstr "シェルを終了するとシステムは自動的に再起動します。"

#: pyanaconda/rescue.py:362
#, python-format
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions. %s"
msgstr ""
"システムに損傷のあるファイルシステムがありマウントしないことを選択していま"
"す。<enter> を押すとシェルが起動され、ファイルシステムの検証(fsck)や パーティ"
"ションのマウント(mount)を行うことができます。 %s"

#: pyanaconda/rescue.py:372
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""
"システムを %(rootPath)s 下にマウントしています。\n"
"\n"
"シェルを起動するには <enter>キーを押してください。そのシステムを root 環境と"
"して利用するには次のコマンドを実行します:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"

#: pyanaconda/rescue.py:443
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell."
msgstr ""
"システムのいくつか、あるいはすべてをマウントしようと試みている間にエラーが発"
"生しました。いくつかのパーティションは %s の下にマウントされています。\n"
"\n"
"シェルを起動するには [Enter] を押してください。"

#: pyanaconda/rescue.py:451
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr "Linux パーティションがありません。再起動します。\n"

#: pyanaconda/rescue.py:455
msgid " The system will reboot automatically when you exit from the shell."
msgstr " シェルを終了するとシステムは自動的に再起動します。"

#: pyanaconda/rescue.py:459
msgid "Rescue Mode"
msgstr "レスキューモード"

#: pyanaconda/rescue.py:460
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""
"Linux パーティションがありません。<enter>キーを押すとシェルを起動します。%s"

#: pyanaconda/rescue.py:472
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "システムは %s ディレクトリー配下にマウントされます。"

#: pyanaconda/storage_utils.py:50 pyanaconda/storage_utils.py:82
#: pyanaconda/ui/gui/spokes/custom.glade:31
msgid "LVM"
msgstr "LVM"

#: pyanaconda/storage_utils.py:51 pyanaconda/storage_utils.py:83
#: pyanaconda/ui/gui/spokes/custom.glade:39
msgid "LVM Thin Provisioning"
msgstr ""

#: pyanaconda/storage_utils.py:52 pyanaconda/ui/gui/spokes/custom.glade:35
msgid "RAID"
msgstr "RAID"

#: pyanaconda/storage_utils.py:53 pyanaconda/storage_utils.py:80
#: pyanaconda/ui/gui/spokes/custom.glade:23
msgid "Standard Partition"
msgstr "標準パーティション"

#: pyanaconda/storage_utils.py:54 pyanaconda/storage_utils.py:81
msgid "BTRFS"
msgstr "Btrfs"

#: pyanaconda/storage_utils.py:55
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:222
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:715
#: pyanaconda/ui/gui/spokes/lib/resize.glade:152
msgid "Disk"
msgstr "ディスク"

#: pyanaconda/storage_utils.py:66
msgid ""
"The 'swap' area on your computer is used by the operating\n"
"system when running low on memory."
msgstr ""
"ご使用のコンピューターの 'swap' 領域は、メモリー不足の状態になると\n"
"オペレーティングシステムで使用されます。"

#: pyanaconda/storage_utils.py:68
msgid ""
"The 'boot' area on your computer is where files needed\n"
"to start the operating system are stored."
msgstr ""
"ご使用のコンピューターの 'boot' 領域は、オペレーティング\n"
"システムの開始に必要なファイルを収納しています。"

#: pyanaconda/storage_utils.py:70
msgid ""
"The 'root' area on your computer is where core system\n"
"files and applications are stored."
msgstr ""
"ご使用のコンピューターの 'root' 領域は、コアシステムファイルと\n"
"アプリKションを収納しています。"

#: pyanaconda/storage_utils.py:72
msgid ""
"The 'home' area on your computer is where all your personal\n"
"data is stored."
msgstr ""
"ご使用のコンピューターの 'home' 領域は、ユーザーの個人\n"
"データを収納しています。"

#: pyanaconda/storage_utils.py:74
msgid ""
"The BIOS boot partition is required to enable booting\n"
"from GPT-partitioned disks on BIOS hardware."
msgstr ""
"BIOS ハードウェア上で GPT パーティション設定したディスク からの起動を有効にす"
"るには、BIOS のブートパーティションが必要です。"

#: pyanaconda/storage_utils.py:76
msgid ""
"The PReP boot partition is required as part of the\n"
"bootloader configuration on some PPC platforms."
msgstr ""
"数種の PPC プラットフォーム上のブートローダー設定の一部 としてPReP ブートパー"
"ティションが必要です。"

#: pyanaconda/storage_utils.py:180
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"root パーティションが 250 MB 以下になっています。通常、 %s のインストールには"
"小さすぎます。"

#: pyanaconda/storage_utils.py:185
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""
"%s のインストールを継続するために必要な root パーティション (/) が定義されて"
"いません。"

#: pyanaconda/storage_utils.py:199
msgid ""
"This platform requires /boot on a dedicated partition or logical volume.  If "
"you do not want a /boot volume, you must place / on a dedicated non-LVM "
"partition."
msgstr ""
"お使いのプラットフォームでは専用のパーティションまたは論理ボリューム上に /"
"boot を配置する必要があります。あなたが /boot ボリュームを作りたくない場合"
"は、非 LVM 上の専用パーティションに / を配置しなければなりません。"

#: pyanaconda/storage_utils.py:209
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"%(mount)s パーティションの容量が %(size)s MB以下であり、通常の "
"%(productName)s のインストールでの 推奨容量には不足しています。"

#: pyanaconda/storage_utils.py:219
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too small for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"このシステムの %(mount)s パーティションは %(format)s フォーマットには小さすぎ"
"ます。 (指定可能なサイズは %(minSize)d MB から %(maxSize)d MB となっています)"

#: pyanaconda/storage_utils.py:225
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is too large for %(format)s formatting (allowable "
"size is %(minSize)s to %(maxSize)s)"
msgstr ""
"このシステムの %(mount)s パーティションは %(format)s フォーマットには大きすぎ"
"ます。 (指定可能なサイズは %(minSize)d MB から %(maxSize)d MB となっています)"

#: pyanaconda/storage_utils.py:234
#, fuzzy
msgid "No valid bootloader target device found. See below for details."
msgstr "新規デバイスの追加に失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/storage_utils.py:246
msgid "You have not created a bootable partition."
msgstr "ブート可能なパーティションが作成されていません。"

#: pyanaconda/storage_utils.py:268
#, fuzzy
msgid ""
"Your BIOS-based system needs a special partition to boot from a GPT disk "
"label. To continue, please create a 1MiB 'biosboot' type partition."
msgstr ""
"ご使用の BIOS ベースのシステムは %s の新規ディスクラベル形式 (GPT) で起動する"
"には、特別なパーティションが必要です。継続するには、1MB の  'BIOS ブート' タ"
"イプのパーティションを作成してください。"

#: pyanaconda/storage_utils.py:279
#, fuzzy, python-format
msgid ""
"You have not specified a swap partition.  %(requiredMem)s of memory is "
"required to continue installation without a swap partition, but you only "
"have %(installedMem)s."
msgstr ""
"swap パーティションが未指定です。swap パーティションなしでのインストール作業"
"の継続には %(requiredMem)s のメモリーが必要となっています。しかし、お使いのシ"
"ステムには %(installedMem)s MB しか搭載されていません。"

#: pyanaconda/storage_utils.py:286
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"swap パーティションが未指定です。すべての場合において必ず必要というわけではあ"
"りませんが、ほとんどのインストールにおいて明らかに性能を向上させます。"

#: pyanaconda/storage_utils.py:293
msgid ""
"At least one of your swap devices does not have a UUID, which is common in "
"swap space created using older versions of mkswap. These devices will be "
"referred to by device path in /etc/fstab, which is not ideal since device "
"paths can change under a variety of circumstances. "
msgstr ""
"swap デバイスの少なくとも1つは UUID を持っていなければなりません。これは旧"
"バージョンの mkswap コマンドを使用して作成した swap 領域ではよくあります。こ"
"れらのデバイスは /etc/fstab のデバイスパスによって参照されます。デバイスパス"
"による参照はさまざまな状況により変更されてしまうため理想的な方法ではありませ"
"ん。"

#: pyanaconda/storage_utils.py:304
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""
"このマウントポイントは正しくありません。%s ディレクトリーは / ファイルシステ"
"ム上になければなりません。"

#: pyanaconda/storage_utils.py:309
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "マウントポイント %s は linux ファイルシステム上にある必要があります。"

#: pyanaconda/storage_utils.py:337
#, python-format
msgid "LUKS device %s has no encryption key"
msgstr ""

#: pyanaconda/text.py:99
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/text.py:106
#, fuzzy
msgid "Passphrase"
msgstr "パスフレーズ:"

#: pyanaconda/vnc.py:75
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr "%(productName)s %(productVersion)s のインストール"

#: pyanaconda/vnc.py:132
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "ホスト %(name)s 上の %(productName)s %(productVersion)s インストール"

#: pyanaconda/vnc.py:151
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "ホスト %s 上の vnc クライアントに接続を試行中..."

#: pyanaconda/vnc.py:165
msgid "Connected!"
msgstr "接続しました!"

#: pyanaconda/vnc.py:168
msgid "Will try to connect again in 15 seconds..."
msgstr "15 秒後に再接続を試みます..."

#: pyanaconda/vnc.py:174
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "%d 回接続を試みた後に断念します。\n"

#: pyanaconda/vnc.py:185
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""
"インストールを開始するには、ご使用の vnc クライアントを %s に手動で接続してく"
"ださい。"

#: pyanaconda/vnc.py:187
#, python-format
msgid ""
"Please manually connect your vnc client to <IP ADDRESS>:%s to begin the "
"install. Switch to the shell (Ctrl-B 2) and run 'ip addr' to find the <IP "
"ADDRESS>."
msgstr ""

#: pyanaconda/vnc.py:192
msgid "Starting VNC..."
msgstr "VNC を起動中..."

#: pyanaconda/vnc.py:234
msgid "The VNC server is now running."
msgstr "VNC サーバーが稼動しています。"

#: pyanaconda/vnc.py:238
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""
"\n"
"\n"
"Listen モードの VNC Viewer に接続するよう選択しました。\n"
"これは、パスワードを必要としません。もしパスワードを設定すると、\n"
"VNC Viewew への接続失敗の原因になります。\n"
"\n"

#: pyanaconda/vnc.py:243
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"警告!!! VNC サーバーがパスワードなしで実行されています!\n"
"サーバーの安全性を確保するには、vncpassword=<password>\n"
"ブートオプションを使うことができます。\n"
"\n"

#: pyanaconda/vnc.py:247
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""
"\n"
"\n"
"パスワードを使って VNC を実行するように選択しました。\n"
"\n"

#: pyanaconda/vnc.py:249
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""
"\n"
"\n"
"不明なエラーが発生しました。作業を中断します。\n"

#: pyanaconda/vnc.py:269
#, fuzzy
msgid ""
"VNC password must be six to eight characters long.\n"
"Please enter a new one, or leave blank for no password."
msgstr ""
"入力された VNC パスワードは最低限の 6文字に足りません。\n"
"新しいパスワードを入力して下さい。パスワード不要なら空白にします。"

#: pyanaconda/installclasses/fedora.py:29
msgid "_Fedora"
msgstr "Fedora(_F)"

#: pyanaconda/installclasses/rhel.py:29
msgid "Red Hat Enterprise Linux"
msgstr "Red Hat Enterprise Linux"

#: pyanaconda/packaging/__init__.py:1130
#, fuzzy
msgid "Failed to set up installation source"
msgstr "インストールソースの設定に失敗しました"

#: pyanaconda/packaging/__init__.py:1131
#, fuzzy
msgid "Error downloading package metadata"
msgstr "パッケージメタデータをダウンロード中にエラー..."

#: pyanaconda/packaging/__init__.py:1132
msgid "No installation source available"
msgstr "インストールソースは利用できません"

#: pyanaconda/packaging/livepayload.py:98
#: pyanaconda/packaging/livepayload.py:119
msgid "Installing software"
msgstr "ソフトウェアのインストール中…"

#: pyanaconda/packaging/livepayload.py:220
#: pyanaconda/packaging/livepayload.py:229
#, python-format
msgid "Downloading %(url)s (%(pct)d%%)"
msgstr ""

#: pyanaconda/packaging/livepayload.py:358
msgid "Checking image checksum"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1207
#, python-format
msgid "CmdlineError: Missing package: %s"
msgstr ""

#: pyanaconda/packaging/yumpayload.py:1352
#: pyanaconda/packaging/dnfpayload.py:493
msgid "Starting package installation process"
msgstr "パッケージインストールプロセスを開始しています"

#: pyanaconda/packaging/yumpayload.py:1388
#: pyanaconda/packaging/dnfpayload.py:522
msgid "Preparing transaction from installation source"
msgstr "インストールソースからのトランザクションを準備中。"

#: pyanaconda/packaging/yumpayload.py:1389
msgid "Installing"
msgstr "インストール中"

#: pyanaconda/packaging/dnfpayload.py:146
#, python-format
msgid ""
"Downloading %(total_files)s RPMs, %(downloaded)s / %(total_size)s "
"(%(percent)d%%) done."
msgstr ""

#: pyanaconda/packaging/dnfpayload.py:510
#, fuzzy
msgid "Downloading packages"
msgstr "パッケージのメタデータをダウンロード中..."

#: pyanaconda/packaging/dnfpayload.py:532
#, fuzzy, python-format
msgid "Installing %s"
msgstr "インストール中"

#: pyanaconda/packaging/rpmostreepayload.py:115
#, python-format
msgid "Starting pull of %(branchName)s from %(source)s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:122
#, fuzzy, python-format
msgid "Preparing deployment of %s"
msgstr "インストールの準備中"

#: pyanaconda/packaging/rpmostreepayload.py:134
#, python-format
msgid "Deployment starting: %s"
msgstr ""

#: pyanaconda/packaging/rpmostreepayload.py:137
#, python-format
msgid "Deployment complete: %s"
msgstr ""

#: pyanaconda/ui/helpers.py:94 pyanaconda/ui/tui/spokes/storage.py:387
msgid "Checking storage configuration..."
msgstr "ストレージ設定をチェック中..."

#: pyanaconda/ui/categories/__init__.py:49
msgid "DEFAULT TITLE"
msgstr "デフォルトのタイトル"

#: pyanaconda/ui/categories/customization.py:32
#, fuzzy
msgid "CUSTOMIZATION"
msgstr "地域設定"

#: pyanaconda/ui/categories/localization.py:31
msgid "LOCALIZATION"
msgstr "地域設定"

#: pyanaconda/ui/categories/software.py:31
msgid "SOFTWARE"
msgstr "ソフトウェア"

#: pyanaconda/ui/categories/system.py:32
#: pyanaconda/ui/gui/spokes/lib/accordion.py:166
msgid "SYSTEM"
msgstr "システム"

#: pyanaconda/ui/categories/user_settings.py:32
msgid "USER SETTINGS"
msgstr "ユーザーの設定"

#: pyanaconda/ui/lib/space.py:37
#, python-format
msgid ""
"Not enough space in filesystems for the current software selection. An "
"additional %s is needed."
msgstr ""
"現在選択しているソフトウェア選択に対してファイルシステムの空き領域が不足して"
"います。追加で %s 必要です。"

#: pyanaconda/ui/lib/entropy.py:58
msgid "The system needs more random data entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:67
msgid "Entropy can be increased by typing randomly on keyboard"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:68
#, python-format
msgid ""
"After %d minutes, the installation will continue regardless of the amount of "
"available entropy"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:92
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%] "
"(%(rem)d %(min)s remaining)"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:95
#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.py:88
msgid "minute"
msgid_plural "minutes"
msgstr[0] ""

#: pyanaconda/ui/lib/entropy.py:102
#, python-format
msgid ""
"Available entropy: %(av_entr)s, Required entropy: %(req_entr)s [%(pct)d %%]"
msgstr ""

#: pyanaconda/ui/lib/entropy.py:107
msgid "Enough entropy gathered, please stop typing."
msgstr ""

#: pyanaconda/ui/lib/entropy.py:110
#, python-format
msgid "Giving up, time (%d minutes) ran out."
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:35
#, fuzzy
msgid "Installation"
msgstr "インストールハブ"

#: pyanaconda/ui/tui/hubs/summary.py:51
msgid "Starting automated install"
msgstr "自動インストールの開始"

#: pyanaconda/ui/tui/hubs/summary.py:82
msgid "The following mandatory spokes are not completed:"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:90
#, fuzzy
msgid ""
"  Please make your choice from above ['q' to quit | 'b' to begin "
"installation |\n"
"  'r' to refresh]: "
msgstr "上記の ['q' 終了 | 'c' 継続] から選択して下さい:"

#. TRANSLATORS: 'b' to begin installation
#. TRANSLATORS: 'b' to go back
#: pyanaconda/ui/tui/hubs/summary.py:104
#: pyanaconda/ui/tui/spokes/langsupport.py:117
msgid "b"
msgstr ""

#: pyanaconda/ui/tui/hubs/summary.py:107
#: pyanaconda/ui/tui/hubs/__init__.py:114
msgid "Please complete all spokes before continuing"
msgstr "すべてのスポークを完了してから継続して下さい。"

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/summary.py:118
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/hubs/__init__.py:41
msgid "Default HUB title"
msgstr "デフォルトのハブタイトル"

#. TRANSLATORS: 'c' to continue
#: pyanaconda/ui/tui/hubs/__init__.py:111
#: pyanaconda/ui/tui/simpleline/base.py:456
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "c"
msgstr "c"

#: pyanaconda/ui/tui/tuiobject.py:30 pyanaconda/ui/gui/main.glade:72
msgid "Error"
msgstr "エラー"

#: pyanaconda/ui/tui/tuiobject.py:51
msgid "Press enter to exit."
msgstr "enter を押して終了します。"

#: pyanaconda/ui/tui/tuiobject.py:61
msgid "Question"
msgstr "質問"

#: pyanaconda/ui/tui/tuiobject.py:84
msgid "Please respond 'yes' or 'no': "
msgstr "「はい」か「いいえ」で答えて下さい: "

#: pyanaconda/ui/tui/tuiobject.py:87
msgid "yes"
msgstr "はい"

#: pyanaconda/ui/tui/tuiobject.py:92
msgid "no"
msgstr "いいえ"

#: pyanaconda/ui/tui/simpleline/base.py:87
msgid "Do you really want to quit?"
msgstr "本当に終了しますか？"

#. TRANSLATORS: 'r' to refresh
#: pyanaconda/ui/tui/simpleline/base.py:450
msgctxt "TUI|Spoke Navigation"
msgid "r"
msgstr ""

#. TRANSLATORS: 'q' to quit
#: pyanaconda/ui/tui/simpleline/base.py:462
#, fuzzy
msgctxt "TUI|Spoke Navigation"
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/simpleline/base.py:569
msgid "Press ENTER to continue"
msgstr ""

#: pyanaconda/ui/tui/simpleline/base.py:619
#, fuzzy
msgid ""
"  Please make your choice from above ['q' to quit | 'c' to continue |\n"
"  'r' to refresh]: "
msgstr "上記の ['q' 終了 | 'c' 継続] から選択して下さい:"

#: pyanaconda/ui/tui/spokes/askvnc.py:39
msgid "VNC"
msgstr "VNC"

#: pyanaconda/ui/tui/spokes/askvnc.py:55
#, fuzzy
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"installation or continue with a text mode installation?"
msgstr ""
"ご使用のマシンで X を起動できませんでした。VNC を起動して別のコンピューターか"
"らこのコンピューターに接続しグラフィカルインストールを行ないますか、あるいは"
"テキストモードのインストールで続行しますか?"

#: pyanaconda/ui/tui/spokes/askvnc.py:103
msgid "q"
msgstr "q"

#: pyanaconda/ui/tui/spokes/askvnc.py:115
msgid "VNC Password"
msgstr "VNC パスワード"

#: pyanaconda/ui/tui/spokes/askvnc.py:123
#, fuzzy
msgid ""
"Please provide VNC password (must be six to eight characters long).\n"
"You will have to type it twice. Leave blank for no password"
msgstr ""
"VNC パスワードを記入してください。2度記入して下さい。\n"
"パスワード不要なら空白にします。"

#: pyanaconda/ui/tui/spokes/askvnc.py:142
msgid "Password: "
msgstr "パスワード: "

#: pyanaconda/ui/tui/spokes/askvnc.py:143
msgid "Password (confirm): "
msgstr "パスワード(確定): "

#: pyanaconda/ui/tui/spokes/askvnc.py:146
msgid "Passwords do not match!"
msgstr "パスワードが一致しません！"

#: pyanaconda/ui/tui/spokes/askvnc.py:149
msgid "The password must be at least six characters long."
msgstr "パスワードは最低 6 文字以上でなければなりません。"

#: pyanaconda/ui/tui/spokes/askvnc.py:153
#, fuzzy
msgid "The password cannot be more than eight characters long."
msgstr "パスワードは最低 6 文字以上でなければなりません。"

#: pyanaconda/ui/tui/spokes/langsupport.py:40
#, fuzzy
msgid "Language settings"
msgstr "タイムゾーンの設定"

#: pyanaconda/ui/tui/spokes/langsupport.py:70
#, fuzzy
msgid "Language is not set."
msgstr "タイムゾーンは設定されていません"

#: pyanaconda/ui/tui/spokes/langsupport.py:80
#, fuzzy
msgid "Available locales"
msgstr "使用できる地域"

#: pyanaconda/ui/tui/spokes/langsupport.py:83
#, fuzzy
msgid "Available languages"
msgstr "使用できる地域"

#: pyanaconda/ui/tui/spokes/langsupport.py:125
msgid ""
"Please select language support to install.\n"
"[b to language list, c to continue, q to quit]: "
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:43
#, fuzzy
msgid "Network configuration"
msgstr "ネットワーク構成を書き込み中"

#: pyanaconda/ui/tui/spokes/network.py:94
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) disconnected\n"
msgstr "有線 (%s) は接続されています"

#: pyanaconda/ui/tui/spokes/network.py:99
#, fuzzy, python-format
msgid "Wired (%(interface_name)s) connected\n"
msgstr "有線 (%s) は接続されています"

#: pyanaconda/ui/tui/spokes/network.py:111
#, python-format
msgid " IPv4 Address: %(addr)s Netmask: %(netmask)s Gateway: %(gateway)s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:113
#, python-format
msgid " DNS: %s\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:120
#, python-format
msgid " IPv6 Address: %(addr)s/%(prefix)d\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:140
#, fuzzy, python-format
msgid "Hostname: %s\n"
msgstr "ホスト名:"

#: pyanaconda/ui/tui/spokes/network.py:152
#, fuzzy
msgid "Set hostname"
msgstr "ホスト名"

#: pyanaconda/ui/tui/spokes/network.py:154
#, python-format
msgid "Configure device %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:173
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:225
#: pyanaconda/ui/gui/spokes/network.glade:2236
msgid "Hostname"
msgstr "ホスト名"

#: pyanaconda/ui/tui/spokes/network.py:191
msgid ""
"Configuration not saved: gateway or netmask missing in static configuration"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:207
msgid "Can't apply configuration, device activation failed."
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:234
#: pyanaconda/ui/gui/spokes/network.py:1378
#: pyanaconda/ui/gui/spokes/network.py:1448
#, python-format
msgid "Hostname is not valid: %s"
msgstr "ホスト名は無効です: %s"

#: pyanaconda/ui/tui/spokes/network.py:258
#, fuzzy
msgid "Device configuration"
msgstr "設定完了(_F)"

#: pyanaconda/ui/tui/spokes/network.py:262
#, python-format
msgid "IPv4 address or %s for DHCP"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:264
msgid "IPv4 netmask"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:265
#, fuzzy
msgid "IPv4 gateway"
msgstr "ゲートウェイ"

#: pyanaconda/ui/tui/spokes/network.py:266
#, python-format
msgid ""
"IPv6 address or %(auto)s for automatic, %(dhcp)s for DHCP, %(ignore)s to "
"turn off"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:269
msgid "IPv6 default gateway"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:270
msgid "Nameservers (comma separated)"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:271
msgid "Connect automatically after reboot"
msgstr ""

#: pyanaconda/ui/tui/spokes/network.py:272
#, fuzzy
msgid "Apply configuration in installer"
msgstr "ファイルシステム設定にタイプがありません。"

#: pyanaconda/ui/tui/spokes/network.py:287
#, fuzzy, python-format
msgid "Configuring device %s."
msgstr "インストール済みのシステムを構成"

#: pyanaconda/ui/tui/spokes/password.py:32
#, fuzzy
msgid "Root password"
msgstr "root パスワード:"

#: pyanaconda/ui/tui/spokes/password.py:55
msgid "Password is set."
msgstr "パスワードは設定されています"

#: pyanaconda/ui/tui/spokes/password.py:57
msgid "Root account is disabled."
msgstr "root アカウントは無効になっています"

#: pyanaconda/ui/tui/spokes/password.py:59
msgid "Password is not set."
msgstr "パスワードが設定されていません"

#: pyanaconda/ui/tui/spokes/password.py:64
msgid "Please select new root password. You will have to type it twice."
msgstr "新しい root パスワードを選択して下さい。２度記入する必要があります。"

#: pyanaconda/ui/tui/spokes/password.py:70
#: pyanaconda/ui/gui/spokes/password.glade:86
#: pyanaconda/ui/gui/spokes/user.glade:172
#, fuzzy
msgid "Password"
msgstr "パスワード: "

#: pyanaconda/ui/tui/spokes/storage.py:55
msgid "Use All Space"
msgstr "すべての領域を使用する"

#: pyanaconda/ui/tui/spokes/storage.py:56
msgid "Replace Existing Linux system(s)"
msgstr "既存の Linux システムを入れ替える"

#: pyanaconda/ui/tui/spokes/storage.py:57
msgid "Use Free Space"
msgstr "空き領域を使用する"

#: pyanaconda/ui/tui/spokes/storage.py:67
#, fuzzy
msgid "Installation Destination"
msgstr "インストール先"

#: pyanaconda/ui/tui/spokes/storage.py:121
#: pyanaconda/ui/gui/spokes/storage.py:360
msgid "No disks selected"
msgstr "ディスクが選択されていません"

#: pyanaconda/ui/tui/spokes/storage.py:131
#: pyanaconda/ui/gui/spokes/storage.py:372
msgid "Error checking storage configuration"
msgstr "ストレージ設定のチェック中にエラー"

#: pyanaconda/ui/tui/spokes/storage.py:133
#: pyanaconda/ui/gui/spokes/storage.py:374
#, fuzzy
msgid "Warning checking storage configuration"
msgstr "ストレージ設定のチェック中にエラー"

#: pyanaconda/ui/tui/spokes/storage.py:136
#: pyanaconda/ui/gui/spokes/storage.py:376
msgid "Automatic partitioning selected"
msgstr "自動パーティション設定が選択されました"

#: pyanaconda/ui/tui/spokes/storage.py:138
#: pyanaconda/ui/gui/spokes/storage.py:378
msgid "Custom partitioning selected"
msgstr "カスタムパーティション設定が選択されました"

#: pyanaconda/ui/tui/spokes/storage.py:174
#: pyanaconda/ui/gui/spokes/storage.py:602
msgid ""
"No disks detected.  Please shut down the computer, connect at least one "
"disk, and restart to complete installation."
msgstr ""
"ディスク検出がありません。コンピュータをシャットダウンして最低１個のディスク"
"に接続し、再開始するとインストールが完了できます。"

#: pyanaconda/ui/tui/spokes/storage.py:176
#: pyanaconda/ui/gui/spokes/storage.py:604
msgid "No disks selected; please select at least one disk to install to."
msgstr ""
"ディスクが選択されていません。インストール先として少なくともディスクを１つ選"
"択して下さい。"

#: pyanaconda/ui/tui/spokes/storage.py:191
#: pyanaconda/ui/tui/spokes/source.py:364
#: pyanaconda/ui/gui/spokes/source.py:652
#: pyanaconda/ui/gui/spokes/storage.py:553
msgid "Probing storage..."
msgstr "ストレージを捜索中..."

#: pyanaconda/ui/tui/spokes/storage.py:212
#, fuzzy
msgid "Select all"
msgstr "選択(_S)"

#: pyanaconda/ui/tui/spokes/storage.py:305
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs can not be used during installation.\n"
"\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:307
msgid ""
"Warning: All storage changes made using the installer will be lost when you "
"choose to format.\n"
"\n"
"Proceed to run dasdfmt?\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:322
#: pyanaconda/ui/gui/spokes/lib/dasdfmt.py:92
#, python-format
msgid "Formatting /dev/%s. This may take a moment."
msgstr ""

#: pyanaconda/ui/tui/spokes/storage.py:366
msgid "Generating updated storage configuration"
msgstr "更新したストレージ設定を生成中"

#: pyanaconda/ui/tui/spokes/storage.py:371
#: pyanaconda/ui/tui/spokes/storage.py:383
#, python-format
msgid "storage configuration failed: %s"
msgstr "ストレージの設定に失敗しました: %s"

#: pyanaconda/ui/tui/spokes/storage.py:430
msgid "Autopartitioning Options"
msgstr "自動パーティション設定のオプション"

#: pyanaconda/ui/tui/spokes/storage.py:456
#, fuzzy
msgid ""
"Installation requires partitioning of your hard drive. Select what space to "
"use for the install target."
msgstr ""
"インストールにはハードドライブのパーティション設定が必要です。インストール先"
"として使用する領域を選択して下さい。"

#: pyanaconda/ui/tui/spokes/storage.py:495
#, fuzzy
msgid "Partition Scheme Options"
msgstr "パーティションスキーマの設定"

#: pyanaconda/ui/tui/spokes/storage.py:519
#, fuzzy
msgid "Select a partition scheme configuration."
msgstr "パーティションスキーマの設定"

#: pyanaconda/ui/tui/spokes/time_spoke.py:31
msgid "Timezone settings"
msgstr "タイムゾーンの設定"

#: pyanaconda/ui/tui/spokes/time_spoke.py:62
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:521
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:527
#, python-format
msgid "%s timezone"
msgstr "%s タイムゾーン"

#: pyanaconda/ui/tui/spokes/time_spoke.py:64
msgid "Timezone is not set."
msgstr "タイムゾーンは設定されていません"

#: pyanaconda/ui/tui/spokes/time_spoke.py:72
#, python-format
msgid "Available timezones in region %s"
msgstr "地域 %s で使用できるタイムゾーン"

#: pyanaconda/ui/tui/spokes/time_spoke.py:75
msgid "Available regions"
msgstr "使用できる地域"

#: pyanaconda/ui/tui/spokes/time_spoke.py:134
msgid ""
"Please select the timezone.\n"
"Use numbers or type names directly [b to region list, q to quit]: "
msgstr ""
"タイムゾーンを選択して下さい。\n"
"番号かタイプ名を直接選択して下さい\n"
"[b で地域一覧、q で終了]: "

#: pyanaconda/ui/tui/spokes/shell_spoke.py:34
msgid "Shell"
msgstr ""

#: pyanaconda/ui/tui/spokes/shell_spoke.py:49
#, fuzzy
msgid "Start shell"
msgstr "VNC を起動する"

#: pyanaconda/ui/tui/spokes/shell_spoke.py:58
msgid "Exit the shell to continue"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:54
#, fuzzy
msgid "Installation source"
msgstr "インストールの進捗"

#: pyanaconda/ui/tui/spokes/source.py:57
#: pyanaconda/ui/tui/spokes/source.py:106
#: pyanaconda/ui/gui/spokes/source.py:547
#: pyanaconda/ui/gui/spokes/source.glade:759
msgid "Closest mirror"
msgstr "最寄りのミラー"

#: pyanaconda/ui/tui/spokes/source.py:98
#: pyanaconda/ui/gui/spokes/source.py:539
#, python-format
msgid "NFS server %s"
msgstr "NFS サーバー %s"

#: pyanaconda/ui/tui/spokes/source.py:100
#: pyanaconda/ui/gui/spokes/source.py:541
msgid "Local media"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:103
#: pyanaconda/ui/tui/spokes/source.py:117
#: pyanaconda/ui/gui/spokes/source.py:535
msgid "Error setting up software source"
msgstr "ソフトウェアソースの設定でエラー"

#: pyanaconda/ui/tui/spokes/source.py:108
#: pyanaconda/ui/tui/spokes/software.py:87
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:529
#: pyanaconda/ui/gui/spokes/software.py:216
#: pyanaconda/ui/gui/spokes/source.py:549
msgid "Nothing selected"
msgstr "何も選択されていません"

#: pyanaconda/ui/tui/spokes/source.py:119
#: pyanaconda/ui/tui/spokes/software.py:67
#, fuzzy
msgid "Processing..."
msgstr "接続中..."

#: pyanaconda/ui/tui/spokes/source.py:125
#, python-format
msgid "Repo URL set to: %s"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:144
#, fuzzy
msgid "CD/DVD"
msgstr "CD/DVD ドライブ"

#: pyanaconda/ui/tui/spokes/source.py:144
#, fuzzy
msgid "local ISO file"
msgstr "ISO ファイル(_I):"

#: pyanaconda/ui/tui/spokes/source.py:144
#, fuzzy
msgid "Network"
msgstr "ネットワーク時刻同期"

#: pyanaconda/ui/tui/spokes/source.py:148
#, fuzzy
msgid "Choose an installation source type."
msgstr "インストールソースは利用できません"

#: pyanaconda/ui/tui/spokes/source.py:241
#: pyanaconda/ui/tui/spokes/source.py:279
msgid "Specify Repo Options"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:245
msgid "Repo URL"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:283
#, fuzzy
msgid "<server>:/<path>"
msgstr "NFS サーバー %s"

#: pyanaconda/ui/tui/spokes/source.py:284
#, fuzzy
msgid "NFS mount options"
msgstr "NFS マウントオプション:"

#: pyanaconda/ui/tui/spokes/source.py:318
#, fuzzy
msgid "Failed to set up installation source. Check the source address."
msgstr ""
"インストールソースの設定に失敗しました。レポジトリの url を確認して下さい"

#: pyanaconda/ui/tui/spokes/source.py:326
#, fuzzy
msgid "Select device containing the ISO file"
msgstr "/boot を含むデバイス"

#: pyanaconda/ui/tui/spokes/source.py:385
msgid "No mountable devices found"
msgstr ""

#: pyanaconda/ui/tui/spokes/source.py:413
#, fuzzy
msgid "Select an ISO to use as install source"
msgstr "インストールソースの設定に失敗しました"

#: pyanaconda/ui/tui/spokes/source.py:442
msgid "No *.iso files found in device root folder"
msgstr ""

#: pyanaconda/ui/tui/spokes/software.py:39
#, fuzzy
msgid "Software selection"
msgstr "ソフトウェアの選択を修正(_M)"

#: pyanaconda/ui/tui/spokes/software.py:65
#: pyanaconda/ui/gui/spokes/software.py:200
msgid "Error checking software selection"
msgstr "ソフトウェアの選択をチェック中にエラー"

#: pyanaconda/ui/tui/spokes/software.py:69
#: pyanaconda/ui/gui/spokes/software.py:203
msgid "Installation source not set up"
msgstr "インストールソースはセットされていません"

#: pyanaconda/ui/tui/spokes/software.py:71
#: pyanaconda/ui/gui/spokes/software.py:206
msgid "Source changed - please verify"
msgstr "ソースが変更されました - ご確認ください"

#: pyanaconda/ui/tui/spokes/software.py:86
#: pyanaconda/ui/gui/spokes/software.py:214
msgid "Custom software selected"
msgstr "カスタムソフトウェアが選択されています"

#: pyanaconda/ui/tui/spokes/software.py:111
#, fuzzy
msgid "Installation source needs to be set up first."
msgstr "インストールソースはセットされていません"

#: pyanaconda/ui/tui/spokes/software.py:121
#: pyanaconda/ui/tui/spokes/software.py:131
#, fuzzy
msgid "Base environment"
msgstr "使用環境を選択します"

#: pyanaconda/ui/tui/spokes/user.py:37
#, fuzzy
msgid "User creation"
msgstr "認証の使用(_U)"

#: pyanaconda/ui/tui/spokes/user.py:114 pyanaconda/ui/gui/spokes/user.py:347
msgid "No user will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:116
#, fuzzy
msgid "You must set a password"
msgstr "パスワードを入力してから確認を行ってください。"

#: pyanaconda/ui/tui/spokes/user.py:118 pyanaconda/ui/gui/spokes/user.py:349
#, python-format
msgid "Administrator %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/user.py:120 pyanaconda/ui/gui/spokes/user.py:351
#, python-format
msgid "User %s will be created"
msgstr ""

#: pyanaconda/ui/tui/spokes/warnings.py:36
#, fuzzy
msgid "Warnings"
msgstr "警告"

#: pyanaconda/ui/tui/spokes/warnings.py:44
#: pyanaconda/ui/gui/spokes/welcome.glade:518
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: pyanaconda/ui/tui/spokes/progress.py:36
#, fuzzy
msgid "Progress"
msgstr "進行中"

#: pyanaconda/ui/tui/spokes/progress.py:136
msgid "\tInstallation complete.  Press return to quit"
msgstr "» インストールは完了しました。改行ボタンを押して終了します"

#: pyanaconda/ui/tui/spokes/__init__.py:48
msgid "Default spoke title"
msgstr "デフォルトのスポークタイトル"

#: pyanaconda/ui/tui/spokes/__init__.py:60
msgid "testing status..."
msgstr "状態をテスト中..."

#: pyanaconda/ui/tui/spokes/__init__.py:102
#, fuzzy
msgid "New value"
msgstr "新しいデバイス"

#: pyanaconda/ui/tui/spokes/__init__.py:122
#, python-format
msgid "%s: "
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:123
#, fuzzy, python-format
msgid "%s (confirm): "
msgstr "パスワード(確定): "

#: pyanaconda/ui/tui/spokes/__init__.py:126
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:146
#, fuzzy, python-format
msgid ""
"You have provided a weak password: %s\n"
"Would you like to use it anyway?"
msgstr "脆弱なパスワードです: %s　それでも使用する場合は完了を押します。"

#: pyanaconda/ui/tui/spokes/__init__.py:149
#, fuzzy
msgid ""
"You have provided a weak password.\n"
"Would you like to use it anyway?"
msgstr "脆弱なパスワードです: %s　それでも使用する場合は完了を押します。"

#: pyanaconda/ui/tui/spokes/__init__.py:157
msgid ""
"You have provided a password containing non-ASCII characters.\n"
"You may not be able to switch between keyboard layouts to login.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:163
#, python-format
msgid "Enter new value for '%s' and press enter\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:171
#, python-format
msgid ""
"You have provided an invalid username: %s\n"
"Tip: Keep your username shorter than 32 characters and do not use spaces.\n"
msgstr ""

#: pyanaconda/ui/tui/spokes/__init__.py:282
#, fuzzy
msgid "Password set."
msgstr "パスワードは設定されています"

#: pyanaconda/ui/gui/__init__.py:475 widgets/src/BaseWindow.c:113
#, fuzzy
msgid "Help!"
msgstr "ヘルプ"

#: pyanaconda/ui/gui/__init__.py:728 widgets/src/StandaloneWindow.c:49
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/__init__.py:743
msgid "_No"
msgstr "いいえ(_N)"

#: pyanaconda/ui/gui/__init__.py:743
msgid "_Yes"
msgstr "はい(_Y)"

#: pyanaconda/ui/gui/hubs/__init__.py:231
msgid ""
"Please complete items marked with this icon before continuing to the next "
"step."
msgstr "このアイコンが付いた項目を完了してから、次のステップへと続けて下さい。"

#: pyanaconda/ui/gui/hubs/progress.py:110
msgid "Complete!"
msgstr "完了しました!"

#: pyanaconda/ui/gui/hubs/progress.py:208
#, fuzzy, python-format
msgid ""
"%s is now successfully installed on your system and ready for you to use!  "
"When you are ready, reboot your system to start using it!"
msgstr ""
"%s は、ここでご使用のシステムに正常にインストールされ、\n"
"使用準備ができました。次に進むために再起動して使用を開始して下さい！"

#: pyanaconda/ui/gui/hubs/progress.py:211
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/hubs/progress.py:219
#, fuzzy, python-format
msgid ""
"%s is now successfully installed, but some configuration still needs to be "
"done.\n"
"Finish it and then click the Finish configuration button please."
msgstr ""
"%s は今ご使用のシステムに正常にインストールされましたが、まだいくつかの設定が"
"必要です。\n"
"それを終了してから「設定の完了」ボタンをクリックして下さい。"

#: pyanaconda/ui/gui/hubs/progress.py:224
#, fuzzy, python-format
msgid ""
"%s is now successfully installed and ready for you to use!\n"
"Go ahead and reboot to start using it!"
msgstr ""
"%s は、ここでご使用のシステムに正常にインストールされ、\n"
"使用準備ができました。次に進むために再起動して使用を開始して下さい！"

#: pyanaconda/ui/gui/spokes/custom.py:106
#, fuzzy, python-format
msgid "Create a new %(container_type)s ..."
msgstr "新規ボリュームグループを作成中 ..."

#: pyanaconda/ui/gui/spokes/custom.py:107
#, python-format
msgid "Create or select %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:109
msgid "Device reconfiguration failed. Click for details."
msgstr "デバイスの設定が失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:111
msgid ""
"Storage configuration reset due to unrecoverable error. Click for details."
msgstr ""
"復元不可能なエラーによりストレージ設定がリセットされました。詳細にはクリック"
"して下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:135
#: pyanaconda/ui/gui/spokes/custom.glade:93
msgid "MANUAL PARTITIONING"
msgstr "手作業のパーティション設定"

#: pyanaconda/ui/gui/spokes/custom.py:342
#, fuzzy, python-format
msgctxt "GUI|Custom Partitioning"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d 個のストレージデバイスが選択されています"

#: pyanaconda/ui/gui/spokes/custom.py:468
#, fuzzy, python-format
msgid ""
"When you create mount points for your %(name)s %(version)s installation, "
"you'll be able to view their details here."
msgstr ""
"%s %s インストール用のマウントポイントを作成する際に、\n"
"ここでその詳細を表示できます。"

#: pyanaconda/ui/gui/spokes/custom.py:628
#, fuzzy, python-format
msgid ""
"/boot/efi must be on a device of type %(oneFsType)s or %(anotherFsType)s"
msgstr "/boot/efi は タイプ %s のデバイス上でなければなりません"

#: pyanaconda/ui/gui/spokes/custom.py:632
#, fuzzy, python-format
msgid "%(fs)s must be on a device of type %(type)s"
msgstr "%s はタイプ %s のデバイス上でなければなりません"

#: pyanaconda/ui/gui/spokes/custom.py:635
#: pyanaconda/ui/gui/spokes/custom.py:637
#, python-format
msgid "%s cannot be encrypted"
msgstr "%s は暗号化できません"

#: pyanaconda/ui/gui/spokes/custom.py:639
msgid "You must create a new filesystem on the root device."
msgstr "新しいファイルシステムは root デバイス上に作成する必要があります。"

#: pyanaconda/ui/gui/spokes/custom.py:644
#, fuzzy, python-format
msgid "Device does not support RAID level selection %s."
msgstr "タイプ %s のデバイスには有効な RAID レベルを選択する必要があります。"

#: pyanaconda/ui/gui/spokes/custom.py:746
msgid "Device resize request failed. Click for details."
msgstr "デバイスのサイズ変更要求は失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:817
msgid "Device reformat request failed. Click for details."
msgstr ""
"デバイスの再フォーマット要求は失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:1187
#, python-format
msgid "Specified name %s already in use."
msgstr "指定された名前 %s はすでに存在します。"

#: pyanaconda/ui/gui/spokes/custom.py:1345
#, fuzzy
msgid "No disks assigned"
msgstr "ディスクが選択されていません"

#: pyanaconda/ui/gui/spokes/custom.py:1351
#, python-format
msgctxt "GUI|Custom Partitioning|Devices"
msgid " and %d other"
msgid_plural " and %d others"
msgstr[0] ""

#: pyanaconda/ui/gui/spokes/custom.py:1442
msgid ""
"The space available to this mountpoint can be changed by modifying the "
"volume below."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:1444
msgid "This file system may not be resized."
msgstr "このファイルシステムはサイズ変更できません。"

#: pyanaconda/ui/gui/spokes/custom.py:1498
#, fuzzy
msgid ""
"Error checking storage configuration.  Click for details or press Done again "
"to continue."
msgstr "ストレージ設定のチェックでエラー発生。クリックして詳細を見ます。"

#: pyanaconda/ui/gui/spokes/custom.py:1500
#, fuzzy
msgid ""
"Warning checking storage configuration.  Click for details or press Done "
"again to continue."
msgstr "ストレージ設定のチェックでエラー発生。クリックして詳細を見ます。"

#: pyanaconda/ui/gui/spokes/custom.py:1606
#, fuzzy, python-format
msgid "Added new %(type)s to existing container %(name)s."
msgstr "新しい %s をすでに存在するコンテナー %s へ追加しました。"

#: pyanaconda/ui/gui/spokes/custom.py:1618
msgid "Failed to add new device. Click for details."
msgstr "新規デバイスの追加に失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:1624
msgid "Invalid partition size set. Use a valid integer."
msgstr ""
"無効なパーティション容量が設定されました。有効な範囲の整数を使用してくださ"
"い。"

#: pyanaconda/ui/gui/spokes/custom.py:1732
msgid "Device removal request failed. Click for details."
msgstr "デバイスの除去要求は失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:1965
#, python-format
msgid "Volume Group name %s is already in use. Not saving changes."
msgstr "ボリュームグループ名 %s はすでに使われています。変更は保存されません。"

#: pyanaconda/ui/gui/spokes/custom.py:1996
#, fuzzy, python-format
msgid "(%s free)"
msgstr "fs free"

#: pyanaconda/ui/gui/spokes/custom.py:2133
#, fuzzy, python-format
msgid ""
"This Software RAID array is missing %(missingMembers)d of %(totalMembers)d "
"member partitions. You can remove it or select a different device."
msgstr ""
"ソフトウェアRAIDアレイの %d / %d のメンバーパーティションが見つかりません。削"
"除するか他のデバイスを選択してください。"

#: pyanaconda/ui/gui/spokes/custom.py:2139
#, fuzzy, python-format
msgid ""
"This LVM Volume Group is missing %(missingPVs)d of %(totalPVs)d physical "
"volumes. You can remove it or select a different device."
msgstr ""
"LVM ボリュームグループの %d / %d の物理ボリュームが見つかりません。削除するか"
"他のデバイスを選択してください。"

#: pyanaconda/ui/gui/spokes/custom.py:2222
msgid "No disks selected."
msgstr "ディスクが選択されていません。"

#: pyanaconda/ui/gui/spokes/custom.py:2228
msgid "Not enough free space on selected disks."
msgstr "選択したディスクには十分な空き領域がありません。"

#: pyanaconda/ui/gui/spokes/custom.py:2234
#: pyanaconda/ui/gui/spokes/custom.py:2260
msgid "Automatic partitioning failed. Click for details."
msgstr "自動パーティション設定は失敗しました。詳細にはクリックして下さい。"

#: pyanaconda/ui/gui/spokes/custom.py:2463
msgid ""
"Continuing with this action will reset all your partitioning selections to "
"their current on-disk state."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2471
#, fuzzy
msgid "_Reset selections"
msgstr "ソフトウェアの選択を修正(_M)"

#: pyanaconda/ui/gui/spokes/custom.py:2471
msgid "_Preserve current selections"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.py:2546
msgid "Failed to unlock encrypted block device. Click for details"
msgstr ""
"暗号化されたブロックデバイスのアンロックに失敗しました。詳細にはクリックして"
"下さい。"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:392
#, fuzzy
msgctxt "GUI|Spoke"
msgid "DATE & _TIME"
msgstr "日付と時刻"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:512
msgid "Restoring hardware time..."
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:523
msgid "Invalid timezone"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:695
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:709
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:891
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:725
msgid "PM"
msgstr "PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:697
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:706
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:890
#: pyanaconda/ui/gui/spokes/datetime_spoke.py:893
msgid "AM"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1040
msgid "You need to set up networking first if you want to use NTP"
msgstr "NTP を使用する場合は、ネットワークを最初に設定する必要があります。"

#: pyanaconda/ui/gui/spokes/datetime_spoke.py:1045
msgid "You have no working NTP server configured"
msgstr "稼働できる NTP サーバーの設定がありません。"

#: pyanaconda/ui/gui/spokes/filter.py:392
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_INSTALLATION DESTINATION"
msgstr "インストール先"

#: pyanaconda/ui/gui/spokes/filter.py:494
#, fuzzy, python-format
msgctxt "GUI|Installation Destination|Filter"
msgid "%d _storage device selected"
msgid_plural "%d _storage devices selected"
msgstr[0] "%d 個のストレージデバイスが選択されています"

#: pyanaconda/ui/gui/spokes/keyboard.py:48
#, python-format
msgid "%s to switch layouts."
msgstr "%s レイアウトの切り替え"

#: pyanaconda/ui/gui/spokes/keyboard.py:272
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_KEYBOARD"
msgstr "キーボード"

#: pyanaconda/ui/gui/spokes/keyboard.py:359
msgid "Testing layouts configuration not available."
msgstr "レイアウト設定のテストは出来ません。"

#: pyanaconda/ui/gui/spokes/keyboard.py:367
#, fuzzy
msgid "Getting list of layouts..."
msgstr "状態をテスト中..."

#: pyanaconda/ui/gui/spokes/keyboard.py:422
msgid "Layout switching not configured."
msgstr "レイアウトの切り替えは設定されていません。"

#: pyanaconda/ui/gui/spokes/langsupport.py:54
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_LANGUAGE SUPPORT"
msgstr "言語"

#: pyanaconda/ui/gui/spokes/network.py:85
msgid "Status unknown (missing)"
msgstr "状態は不明です (情報欠如)"

#: pyanaconda/ui/gui/spokes/network.py:88
msgid "Status unknown"
msgstr "状態は不明です"

#: pyanaconda/ui/gui/spokes/network.py:90
msgid "Unmanaged"
msgstr "管理されていません"

#: pyanaconda/ui/gui/spokes/network.py:93
#: pyanaconda/ui/gui/spokes/network.py:97
msgid "Unavailable"
msgstr "利用できません"

#: pyanaconda/ui/gui/spokes/network.py:95
msgid "Firmware missing"
msgstr "ファームウェアがありません"

#: pyanaconda/ui/gui/spokes/network.py:101
msgid "Cable unplugged"
msgstr "ケーブルが外れています"

#: pyanaconda/ui/gui/spokes/network.py:103
msgid "Disconnected"
msgstr "切断されています"

#: pyanaconda/ui/gui/spokes/network.py:108
msgid "Connecting"
msgstr "接続しています"

#: pyanaconda/ui/gui/spokes/network.py:110
msgid "Authentication required"
msgstr "認証が必要です"

#: pyanaconda/ui/gui/spokes/network.py:112
msgid "Connected"
msgstr "接続済みです"

#: pyanaconda/ui/gui/spokes/network.py:114
msgid "Disconnecting"
msgstr "切断しています"

#: pyanaconda/ui/gui/spokes/network.py:116
msgid "Connection failed"
msgstr "接続に失敗しました"

#: pyanaconda/ui/gui/spokes/network.py:298
msgid "Ethernet"
msgstr "イーサネット"

#: pyanaconda/ui/gui/spokes/network.py:299
msgid "Wireless"
msgstr "ワイヤーレス"

#: pyanaconda/ui/gui/spokes/network.py:300
#: pyanaconda/ui/gui/spokes/network.glade:15
msgid "Bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:301
#: pyanaconda/ui/gui/spokes/network.glade:23
msgid "Vlan"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:302
#: pyanaconda/ui/gui/spokes/network.glade:19
msgid "Team"
msgstr ""

#. TRANSLATORS: ethernet cable is unplugged
#: pyanaconda/ui/gui/spokes/network.py:692
msgid "unplugged"
msgstr "外れています"

#: pyanaconda/ui/gui/spokes/network.py:782
#: pyanaconda/ui/gui/spokes/network.glade:536
#: pyanaconda/ui/gui/spokes/network.glade:837
msgid "IPv4 Address"
msgstr "IPv4 アドレス"

#: pyanaconda/ui/gui/spokes/network.py:783
#: pyanaconda/ui/gui/spokes/network.glade:512
#: pyanaconda/ui/gui/spokes/network.glade:849
#: pyanaconda/ui/gui/spokes/network.glade:1417
msgid "IPv6 Address"
msgstr "IPv6 アドレス"

#: pyanaconda/ui/gui/spokes/network.py:785
#: pyanaconda/ui/gui/spokes/network.py:787
#: pyanaconda/ui/gui/spokes/network.glade:1405
msgid "IP Address"
msgstr "IP アドレス"

#: pyanaconda/ui/gui/spokes/network.py:876
#, python-format
msgid "%d Mb/s"
msgstr "%d Mb/s"

#: pyanaconda/ui/gui/spokes/network.py:1080
msgid "WEP"
msgstr "WEP"

#: pyanaconda/ui/gui/spokes/network.py:1083
msgid "WPA"
msgstr "WPA"

#: pyanaconda/ui/gui/spokes/network.py:1086
msgid "WPA2"
msgstr "WPA2"

#: pyanaconda/ui/gui/spokes/network.py:1090
msgid "Enterprise"
msgstr "企業"

#: pyanaconda/ui/gui/spokes/network.py:1095 widgets/src/SpokeSelector.c:57
#: pyanaconda/ui/gui/spokes/filter.glade:158
#: pyanaconda/ui/gui/spokes/filter.glade:560
#: pyanaconda/ui/gui/spokes/filter.glade:904
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:532
msgid "None"
msgstr "なし"

#: pyanaconda/ui/gui/spokes/network.py:1219
#: pyanaconda/ui/gui/spokes/network.glade:2438
#, fuzzy
msgid "Authentication required by wireless network"
msgstr "認証が必要です"

#: pyanaconda/ui/gui/spokes/network.py:1220
#, python-format
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network '%(network_id)s'."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1233
#, fuzzy
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Password:"
msgstr "パスワード: "

#: pyanaconda/ui/gui/spokes/network.py:1241
msgctxt "GUI|Network|Secrets Dialog"
msgid "_Key:"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.py:1298
msgctxt "GUI|Spoke"
msgid "_NETWORK & HOSTNAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.py:51
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_ROOT PASSWORD"
msgstr "root パスワード"

#: pyanaconda/ui/gui/spokes/password.py:98
#: pyanaconda/ui/gui/spokes/password.py:99
#, fuzzy
msgid "The password is set."
msgstr "パスワードは設定されています"

#: pyanaconda/ui/gui/spokes/password.py:121
msgid "Root password is set"
msgstr "root パスワードは設定されています"

#: pyanaconda/ui/gui/spokes/password.py:123
msgid "Root account is disabled"
msgstr "root アカウントは無効になっています"

#: pyanaconda/ui/gui/spokes/password.py:125
msgid "Root password is not set"
msgstr "root パスワードが設定されていません"

#: pyanaconda/ui/gui/spokes/software.py:52
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_SOFTWARE SELECTION"
msgstr "ソフトウェアの選択"

#: pyanaconda/ui/gui/spokes/software.py:92
#: pyanaconda/ui/gui/spokes/source.py:54
msgid "Downloading package metadata..."
msgstr "パッケージのメタデータをダウンロード中..."

#: pyanaconda/ui/gui/spokes/software.py:95
msgid "Downloading group metadata..."
msgstr "グループメタデータをダウンロード中"

#: pyanaconda/ui/gui/spokes/software.py:135
#: pyanaconda/ui/gui/spokes/source.py:529
msgid "Checking software dependencies..."
msgstr "ソフトウェアの依存関係をチェック中..."

#: pyanaconda/ui/gui/spokes/software.py:140
msgid "Error checking software dependencies"
msgstr "ソフトウェアの依存関係をチェック中にエラー"

#: pyanaconda/ui/gui/spokes/software.py:356
msgid "Error checking software dependencies.  Click for details."
msgstr "ソフトウェアの依存関係のチェック中にエラー。クリックして詳細をみます。"

#: pyanaconda/ui/gui/spokes/software.py:460
#, fuzzy
msgid ""
"The software marked for installation has the following errors.  This is "
"likely caused by an error with your installation source.  You can quit the "
"installer, change your software source, or change your software selections."
msgstr ""
"インストール用にマークしてある以下のソフトウェアにエラーがあります。これはイ"
"ンストールソースが持つエラーが原因であるようです。これらのパッケージをインス"
"トールから除外するか、インストールソースを変えるか、またはインストーラーを終"
"了することができます。"

#: pyanaconda/ui/gui/spokes/software.py:465
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/spokes/software.py:466
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "_Modify Software Source"
msgstr "ソフトウェア提供元を修正(_M)"

#: pyanaconda/ui/gui/spokes/software.py:467
#, fuzzy
msgctxt "GUI|Software Selection|Error Dialog"
msgid "Modify _Selections"
msgstr "ソフトウェアの選択を修正(_M)"

#: pyanaconda/ui/gui/spokes/source.py:53
msgid "Setting up installation source..."
msgstr "インストールソースをセットアップ中..."

#: pyanaconda/ui/gui/spokes/source.py:83
msgid "Invalid proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:88
#, python-format
msgid "Invalid proxy protocol: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:94
msgid "Extra characters in proxy URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:98
#, fuzzy
msgid "Proxy authentication data duplicated"
msgstr "認証が必要です"

#: pyanaconda/ui/gui/spokes/source.py:231
msgid "This media is good to install from."
msgstr "このメディアは以下からのインストールに適切です。"

#: pyanaconda/ui/gui/spokes/source.py:233
msgid "This media is not good to install from."
msgstr "このメディアは以下からのインストールに不適切です。"

#: pyanaconda/ui/gui/spokes/source.py:354
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_INSTALLATION SOURCE"
msgstr "インストールソース"

#: pyanaconda/ui/gui/spokes/source.py:469
#: pyanaconda/ui/gui/spokes/source.py:668
#, fuzzy
msgid "Failed to set up installation source; check the repo url"
msgstr ""
"インストールソースの設定に失敗しました。レポジトリの url を確認して下さい"

#: pyanaconda/ui/gui/spokes/source.py:533
#, fuzzy
msgid "Error setting up base repository"
msgstr "リポジトリーの設定中にエラー"

#: pyanaconda/ui/gui/spokes/source.py:544
#, fuzzy
msgid "Error setting up ISO file"
msgstr "ソフトウェアソースの設定でエラー"

#: pyanaconda/ui/gui/spokes/source.py:670
#, fuzzy
msgid ""
"Failed to set up installation source; check the repo url and proxy settings"
msgstr ""
"インストールソースの設定に失敗しました。レポジトリの url とプロキシの設定を確"
"認して下さい"

#: pyanaconda/ui/gui/spokes/source.py:699
#, fuzzy, python-format
msgid "Device: %s"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/source.py:700
#, python-format
msgid "Label: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:888
msgid "URL is empty"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:892
msgid "Invalid URL"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:909
msgid "Protocol in URL does not match selected protocol"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:912
#, fuzzy
msgid "NFS server is empty"
msgstr "NFS サーバー %s"

#: pyanaconda/ui/gui/spokes/source.py:919
#, fuzzy
msgid "Invalid host name"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/source.py:922
msgid "Remote directory is required"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:943
msgid "Duplicate repository names."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.py:950
#, fuzzy
msgid "Empty repository name"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/source.py:953
#, fuzzy
msgid "Invalid repository name"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/source.py:959
msgid "Repository name conflicts with internal repository name."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:113
msgid "Please wait... software metadata still loading."
msgstr "少々お待ち下さい... ソフトウェアのメタデータがまだロード中です。"

#: pyanaconda/ui/gui/spokes/storage.py:114
#, python-format
msgid ""
"Your current <a href=\"\" title=\"%(tooltip)s\"><b>%(product)s</b> software "
"selection</a> requires <b>%(total)s</b> of available space, including <b>"
"%(software)s</b> for software and <b>%(swap)s</b> for swap space."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:164
#, fuzzy, python-format
msgid "%s The disks you've selected have the following amounts of free space:"
msgstr ""
"%s\n"
"選択したディスクには、以下の容量の空き領域があります:"

#: pyanaconda/ui/gui/spokes/storage.py:172
#, fuzzy, python-format
msgid ""
"<b>You don't have enough space available to install %s</b>.  You can shrink "
"or remove existing partitions via our guided reclaim space tool, or you can "
"adjust your partitions on your own in the custom partitioning interface."
msgstr ""
"<b>%s をインストールするには利用可能領域が足りませんが、</b>既存パーティショ"
"ンを縮小したり、削除したりして領域を取り戻すことができます。"

#: pyanaconda/ui/gui/spokes/storage.py:196
#, fuzzy, python-format
msgid ""
"%(sw_text)s You don't have enough space available to install <b>%(product)s</"
"b>, even if you used all of the free space available on the selected disks."
msgstr ""
"%s\n"
"選択したディスクで利用できるすべての空き領域を使用しても、<b>%s</b> のインス"
"トールには十分な領域がありません。"

#: pyanaconda/ui/gui/spokes/storage.py:207
#, fuzzy, python-format
msgid ""
"<b>You don't have enough space available to install %(productName)s</b>, "
"even if you used all of the free space available on the selected disks.  You "
"could add more disks for additional space, modify your software selection to "
"install a smaller version of <b>%(productName)s</b>, or quit the installer."
msgstr ""
"選択したディスクで利用できるすべての空き領域を使用しても、<b>%s のインストー"
"ルには十分な領域がありません。</b> 更なる領域の為に他のディスクを追加するか、"
"小さいバージョンの <b>%s</b> をインストールするためにソフトウェア選択を修正す"
"るか、またはインストーラーを終了することができます。"

#: pyanaconda/ui/gui/spokes/storage.py:228
#, fuzzy
msgctxt "GUI|Spoke"
msgid "INSTALLATION _DESTINATION"
msgstr "インストール先"

#: pyanaconda/ui/gui/spokes/storage.py:311
msgid "Saving storage configuration..."
msgstr "ストレージ設定を保存しています..."

#: pyanaconda/ui/gui/spokes/storage.py:317
#: pyanaconda/ui/gui/spokes/storage.py:329
msgid "Failed to save storage configuration..."
msgstr "ストレージ設定の保存に失敗しました..."

#: pyanaconda/ui/gui/spokes/storage.py:365
#: pyanaconda/ui/gui/spokes/storage.py:635
#, fuzzy
msgid "Formatting DASDs"
msgstr "DASD デバイスをフォーマット中..."

#: pyanaconda/ui/gui/spokes/storage.py:499
msgid "Error checking storage configuration.  Click for details."
msgstr "ストレージ設定のチェックでエラー発生。クリックして詳細を見ます。"

#: pyanaconda/ui/gui/spokes/storage.py:501
#, fuzzy
msgid "Warning checking storage configuration.  Click for details."
msgstr "ストレージ設定のチェックでエラー発生。クリックして詳細を見ます。"

#: pyanaconda/ui/gui/spokes/storage.py:537
#, fuzzy, python-format
msgid "%s free"
msgstr "fs free"

#: pyanaconda/ui/gui/spokes/storage.py:584
#, fuzzy, python-format
msgid "%(count)d disk selected; %(capacity)s capacity; %(free)s free"
msgid_plural "%(count)d disks selected; %(capacity)s capacity; %(free)s free"
msgstr[0] "選択されたディスク数 %d; 容量 %s; 空き領域 %s"

#: pyanaconda/ui/gui/spokes/storage.py:672
msgid ""
"You have chosen to skip bootloader installation.  Your system may not be "
"bootable."
msgstr ""
"ブートローダーのインストールを省略するよう選択しています。 システムが起動不可"
"になる可能性があります。"

#: pyanaconda/ui/gui/spokes/storage.py:886
msgid "You'll be able to make space available during custom partitioning."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.py:902
#, fuzzy
msgid ""
"The following errors were encountered when checking your storage "
"configuration.  You can modify your storage layout or quit the installer."
msgstr ""
"ストレージ設定をチェック中に以下のエラーに遭遇しました。ストレージレイアウト"
"を変更するか、またはインストーラーを終了することが出来ます。"

#: pyanaconda/ui/gui/spokes/storage.py:907
#, fuzzy
msgctxt "GUI|Storage|Error Dialog"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/spokes/storage.py:908
#, fuzzy
msgctxt "GUI|Storage|Error Dialog"
msgid "_Modify Storage Layout"
msgstr "ストレージの構成変更(_M)"

#: pyanaconda/ui/gui/spokes/storage.py:921
#, fuzzy
msgid ""
"The following warnings were encountered when checking your storage "
"configuration.  These are not fatal, but you may wish to make changes to "
"your storage layout."
msgstr ""
"ストレージ設定をチェック中に以下のエラーに遭遇しました。ストレージレイアウト"
"を変更するか、またはインストーラーを終了することが出来ます。"

#: pyanaconda/ui/gui/spokes/storage.py:925
#, fuzzy
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/user.py:71
#, python-format
msgid "Invalid group name: %s"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:212
#, fuzzy
msgctxt "GUI|Spoke"
msgid "_USER CREATION"
msgstr "ユーザーの設定"

#: pyanaconda/ui/gui/spokes/user.py:285 pyanaconda/ui/gui/spokes/user.py:286
msgid "The password was set by kickstart."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.py:314
#, fuzzy
msgid "Invalid username"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/user.py:316
msgid "Full name cannot contain colon characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.py:254
#, fuzzy, python-format
msgid "WELCOME TO %(name)s %(version)s."
msgstr "%s %s へようこそ。"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.py:270
#, python-format
msgid ""
"The following nodes were discovered using the iSCSI initiator <b>"
"%(initiatorName)s</b> using the target IP address <b>%(targetAddress)s</b>.  "
"Please select which nodes you wish to log into:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:157
msgid "DATA"
msgstr "データ"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:278
#, fuzzy, python-format
msgid ""
"You haven't created any mount points for your %(product)s %(version)s "
"installation yet.  You can:"
msgstr "%s %s インストール用のマウントポイントをまだ選択していません:"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:287
#, fuzzy
msgid "_Click here to create them automatically."
msgstr "ここをクリックすると自動的に作成します。"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:312
#, fuzzy
msgid "Create new mount points by clicking the '+' button."
msgstr "あるいは、'+' アイコンで以下に新規のマウントポイントを作成します。"

#: pyanaconda/ui/gui/spokes/lib/accordion.py:320
msgid ""
"Or, assign new mount points to existing partitions after selecting them "
"below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/accordion.py:325
msgid "_New mount points will use the following partitioning scheme:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.py:131
#, fuzzy, python-format
msgid ""
"<b>%(count)d disk; %(size)s capacity; %(free)s free space</b> (unpartitioned "
"and in filesystems)"
msgid_plural ""
"<b>%(count)d disks; %(size)s capacity; %(free)s free space</b> "
"(unpartitioned and in filesystems)"
msgstr[0] ""
"<b>ディスク数 %d; 容量%s ; 空き領域 %s</b> (パーティション未設定でファイルシ"
"ステム内)"

#: pyanaconda/ui/gui/spokes/lib/cart.py:190
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Do not install bootloader"
msgstr "ブートローダーをインストールしない(_D)"

#: pyanaconda/ui/gui/spokes/lib/cart.py:193
#: pyanaconda/ui/gui/spokes/lib/cart.glade:183
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Set as Boot Device"
msgstr "起動デバイスとして設定(_S)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:63
#, fuzzy, python-format
msgid ""
"The RAID level you have selected (%(level)s) requires more disks (%(min)d) "
"than you currently have selected (%(count)d)."
msgstr ""
"選択した RAID レベルには現在選択している領域より大きな領域が必要になります。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:67
#, python-format
msgid "CONFIGURE %(container_type)s"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:68
#, fuzzy, python-format
msgid ""
"Please create a name for this %(container_type)s and select at least one "
"disk below."
msgstr ""
"ボリュームグループ名を決めて作成し、以下から少なくても1つのディスクを選択して"
"ください。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:71
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:72
#, fuzzy
msgid "Volume Group"
msgstr "ボリュームグループ:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:73
#, fuzzy
msgid "Volume"
msgstr "列"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:113
msgid "Can not relabel already existing filesystem."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:118
#, fuzzy
msgid "Can not set label on filesystem."
msgstr "マウントポイント %s は linux ファイルシステム上にある必要があります。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:120
msgid "Unacceptable label format for filesystem."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:130
msgid "That mount point is already in use. Try something else?"
msgstr "そのマウントポイントはすでに使用中です。他を試しますか?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:132
msgid "Please enter a valid mountpoint."
msgstr "有効なマウントポイントを記入して下さい"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:134
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:146
msgid "That mount point is invalid. Try something else?"
msgstr "そのマウントポイントは無効です。他を試しますか?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:287
msgid "container"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:359
#, fuzzy, python-format
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "Delete _all other filesystems in the %s root as well."
msgstr "%s root にある他のファイルシステムもすべて削除されます。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:369
#, python-format
msgid "Are you sure you want to delete all of the data on %s?"
msgstr "%s にある全データを本当に削除してもよいですか?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:371
#, fuzzy, python-format
msgid ""
"Are you sure you want to delete all of the data on %s, including snapshots "
"and/or subvolumes?"
msgstr "%s にある全データを本当に削除してもよいですか?"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py:648
#, fuzzy
msgid "Invalid container name"
msgstr "無効なタイムゾーン"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:49
#, fuzzy
msgctxt "GUI|Detailed Error Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/detailederror.py:59
#, fuzzy
msgctxt "GUI|Detailed Error Dialog"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:34
#, python-format
msgid "You have provided a weak passphrase: %s"
msgstr "脆弱なパスフレーズを記入しています: %s"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:35
msgid "Passphrases do not match."
msgstr "パスフレーズが一致しません。"

#: pyanaconda/ui/gui/spokes/lib/passphrase.py:157
msgid "Passphrase contains non-ASCII characters"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:53
msgid "Preserve"
msgstr "保存"

#: pyanaconda/ui/gui/spokes/lib/resize.py:54
msgid "Shrink"
msgstr "縮小"

#: pyanaconda/ui/gui/spokes/lib/resize.py:55
msgid "Delete"
msgstr "削除"

#: pyanaconda/ui/gui/spokes/lib/resize.py:114
msgid "This device contains the installation source."
msgstr "インストールソースが含まれているデバイス"

#: pyanaconda/ui/gui/spokes/lib/resize.py:160
#, fuzzy, python-format
msgid "%(freeSize)s of %(devSize)s"
msgstr "%(device)s 上に %(type)s を作成"

#: pyanaconda/ui/gui/spokes/lib/resize.py:167
#, fuzzy
msgid "Not resizeable"
msgstr "準備できていません"

#: pyanaconda/ui/gui/spokes/lib/resize.py:191
#, fuzzy
msgid "Free space"
msgstr "空き領域を使用する"

#: pyanaconda/ui/gui/spokes/lib/resize.py:213
#, fuzzy
msgid ""
"You can remove existing filesystems you no longer need to free up space for "
"this installation.  Removing a filesystem will permanently delete all of the "
"data it contains."
msgstr ""
"このインストールに十分な空き領域がありません。\n"
"\n"
"必要の無い既存ファイルシステムを削除してこのインストール用に空き領域を得るこ"
"とができます。　\n"
"ファイルシステムを削除するとそこに収納してあるすべてのデータを永久に喪失しま"
"す。"

#: pyanaconda/ui/gui/spokes/lib/resize.py:219
msgid ""
"There is also free space available in pre-existing filesystems.  While it's "
"risky and we recommend you back up your data first, you can recover that "
"free disk space and make it available for this installation below."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.py:229
#, fuzzy, python-format
msgid "<b>%(count)s disk; %(size)s reclaimable space</b> (in filesystems)"
msgid_plural ""
"<b>%(count)s disks; %(size)s reclaimable space</b> (in filesystems)"
msgstr[0] "<b>%s 個のディスク; %s つの獲得可能領域</b> (ファイルシステム内) "

#: pyanaconda/ui/gui/spokes/lib/resize.py:236
#, python-format
msgid "Total selected space to reclaim: <b>%s</b>"
msgstr "選択した獲得する領域合計: <b>%s</b>"

#: pyanaconda/ui/gui/spokes/lib/resize.py:451
#: pyanaconda/ui/gui/spokes/lib/resize.py:456
#: pyanaconda/ui/gui/spokes/lib/resize.glade:269
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "Delete _all"
msgstr "削除"

#: pyanaconda/ui/gui/spokes/lib/resize.py:453
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "Preserve _all"
msgstr "保存"

#: widgets/src/BaseWindow.c:109
msgid "DISTRIBUTION INSTALLATION"
msgstr "ディストリビューションのインストール"

#: widgets/src/BaseWindow.c:110
msgid "SPOKE NAME"
msgstr "スポーク名"

#: widgets/src/BaseWindow.c:111
msgid "PRE-RELEASE / TESTING"
msgstr "プレリリース / テスト中"

#: widgets/src/DiskOverview.c:54
msgid "New Device"
msgstr "新しいデバイス"

#: widgets/src/DiskOverview.c:56 widgets/src/DiskOverview.c:57
msgid "0 MB"
msgstr "0 MB"

#: widgets/src/MountpointSelector.c:52
msgid "0 GB"
msgstr "0 GB"

#: widgets/src/SpokeSelector.c:58
msgid "New Selector"
msgstr "新しいセレクター"

#: widgets/src/SpokeWindow.c:46
msgid "_Done"
msgstr "完了(_D)"

#: widgets/src/StandaloneWindow.c:50
msgid "_Continue"
msgstr "続行(_C)"

#: widgets/src/LayoutIndicator.c:32
#, c-format
msgid "Current layout: '%s'. Click to switch to the next layout."
msgstr ""

#: widgets/src/LayoutIndicator.c:33
#, c-format
msgid "Current layout: '%s'. Add more layouts to enable switching."
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:36
#, fuzzy
msgctxt "GUI|Advanced User"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:50
#, fuzzy
msgctxt "GUI|Advanced User"
msgid "_Save Changes"
msgstr "変更を適用(_A)"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:81
#, fuzzy
msgid "ADVANCED USER CONFIGURATION"
msgstr "ネットワーク設定"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:103
msgctxt "GUI|Advanced User"
msgid "Home _directory:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:138
msgid "User and Group IDs"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:157
msgctxt "GUI|Advanced User"
msgid "Specify a _user ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:173
msgctxt "GUI|Advanced User"
msgid "Specify a _group ID manually:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:231
#, fuzzy
msgid "Group Membership"
msgstr "グループ名"

#: pyanaconda/ui/gui/spokes/advanced_user.glade:259
msgctxt "GUI|Advanced User"
msgid "_Add user to the following groups:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:291
msgid "Example:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:308
msgid "wheel, my-team (1245), project-x (29935)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:342
msgid "Tip:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advanced_user.glade:361
msgid ""
"You may input a comma-separated list of group names and group IDs here. "
"Groups that do not already exist will be created; specify their GID in "
"parentheses. "
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:108
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:122
#, fuzzy
msgctxt "GUI|Date and Time|NTP"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:147
msgid "Add and mark for usage NTP servers"
msgstr "NTP サーバーの使用法を追加してマークする"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:170
#, fuzzy
msgid "New NTP Server"
msgstr "Web サーバー"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:189
msgid "Add NTP Server"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:240
msgid "Working"
msgstr "稼働中"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:248
msgid "Use"
msgstr "使用"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:276
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:490
#, fuzzy
msgid "Configure NTP"
msgstr "設定する(_C)..."

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:297
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:879
msgid "DATE & TIME"
msgstr "日付と時刻"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:347
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_Region:"
msgstr "地域:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:377
#, fuzzy
msgid "Region"
msgstr "地域:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:394
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_City:"
msgstr "都市:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:424
#, fuzzy
msgid "City"
msgstr "都市:"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:452
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_Network Time"
msgstr "ネットワーク名(_N)"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:470
#, fuzzy
msgid "Use Network Time"
msgstr "ネットワーク名"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:557
msgid "Hours"
msgstr ""

#. TRANSLATORS: This is the separator between hours and minutes, like in HH:MM
#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:570
msgid ":"
msgstr ":"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:590
msgid "Minutes"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:608
msgid "Hour Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:626
msgid "Hour Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:644
#, fuzzy
msgid "AM/PM Up"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:662
msgid "Minutes Up"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:680
#, fuzzy
msgid "AM/PM Down"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:698
msgid "Minutes Down"
msgstr ""

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:767
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "24-_hour"
msgstr "24 時間"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:785
#, fuzzy
msgctxt "GUI|Date and Time"
msgid "_AM/PM"
msgstr "AM/PM"

#: pyanaconda/ui/gui/spokes/datetime_spoke.glade:844
msgid "Set Date & Time"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:58
#: pyanaconda/ui/gui/spokes/storage.glade:446
#: pyanaconda/ui/gui/spokes/storage.glade:1026
msgid "INSTALLATION DESTINATION"
msgstr "インストール先"

#: pyanaconda/ui/gui/spokes/filter.glade:131
msgid "Search Results:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:144
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "Search _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:159
msgid "Port / Target / LUN #"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:160
msgid "Target WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:197
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Port:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:222
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_Target:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:248
msgctxt "GUI|Installation Destination|Filter|Search|Port Target LUN"
msgid "_LUN:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:288
msgctxt "GUI|Installation Destination|Filter|Search|WWID"
msgid "_WWID:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:360
#: pyanaconda/ui/gui/spokes/filter.glade:563
#: pyanaconda/ui/gui/spokes/filter.glade:741
msgid "WWID"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:374
#: pyanaconda/ui/gui/spokes/filter.glade:755
#: pyanaconda/ui/gui/spokes/filter.glade:1101
#: pyanaconda/ui/gui/spokes/filter.glade:1351
#: pyanaconda/ui/gui/spokes/lib/cart.glade:145
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:235
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:728
msgid "Capacity"
msgstr "容量"

#: pyanaconda/ui/gui/spokes/filter.glade:388
#: pyanaconda/ui/gui/spokes/filter.glade:562
#: pyanaconda/ui/gui/spokes/filter.glade:784
#: pyanaconda/ui/gui/spokes/filter.glade:906
#: pyanaconda/ui/gui/spokes/filter.glade:1130
#, fuzzy
msgid "Interconnect"
msgstr "接続していません"

#: pyanaconda/ui/gui/spokes/filter.glade:402
msgid "Model"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:416
#: pyanaconda/ui/gui/spokes/filter.glade:1379
msgid "LUN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:430
msgid "Port"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:444
msgid "Target"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:458
#: pyanaconda/ui/gui/spokes/filter.glade:561
#: pyanaconda/ui/gui/spokes/filter.glade:769
#: pyanaconda/ui/gui/spokes/filter.glade:905
#: pyanaconda/ui/gui/spokes/filter.glade:1115
msgid "Vendor"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:479
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "_Clear"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:493
msgctxt "GUI|Installation Destination|Filter|Search"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:515
msgctxt "GUI|Installation Destination|Filter"
msgid "Searc_h"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:546
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:599
msgctxt "GUI|Installation Destination|Filter|Multipath|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:637
msgctxt "GUI|Installation Destination|Filter|Multipath|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:675
msgctxt "GUI|Installation Destination|Filter|Multipath|WWID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:798
msgid "Paths"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:819
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "_Clear"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:833
msgctxt "GUI|Installation Destination|Filter|Multipath"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:858
msgctxt "GUI|Installation Destination|Filter"
msgid "_Multipath Devices"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:890
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "Filter _By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:907
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:261
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:754
msgid "ID"
msgstr "ID"

#: pyanaconda/ui/gui/spokes/filter.glade:943
msgctxt "GUI|Installation Destination|Filter|Other|Vendor"
msgid "Show Only _Devices From:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:981
msgctxt "GUI|Installation Destination|Filter|Other|Interconnect"
msgid "Show Only _Devices With:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1019
msgctxt "GUI|Installation Destination|Filter|Other|ID"
msgid "Show Only _Devices Containing:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1087
msgid "Identifier"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1151
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "_Clear"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1165
msgctxt "GUI|Installation Destination|Filter|Other"
msgid "_Find"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1190
#, fuzzy
msgctxt "GUI|Installation Destination|Filter"
msgid "_Other SAN Devices"
msgstr "選択したデバイス"

#: pyanaconda/ui/gui/spokes/filter.glade:1222
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Filter By:"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1236
#, fuzzy
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Label"
msgstr "<b>ラベル:</b>"

#: pyanaconda/ui/gui/spokes/filter.glade:1247
#, fuzzy
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Go"
msgstr "いいえ(_N)"

#: pyanaconda/ui/gui/spokes/filter.glade:1260
msgctxt "GUI|Installation Destination|Filter|zSeries"
msgid "_Clear"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1323
msgid "CCW"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1337
#: pyanaconda/ui/gui/spokes/lib/summary.glade:152
#, fuzzy
msgid "Type"
msgstr "VPN タイプ"

#: pyanaconda/ui/gui/spokes/filter.glade:1365
msgid "WWPN"
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1411
#, fuzzy
msgctxt "GUI|Installation Destination|Filter"
msgid "_zSeries Devices"
msgstr "選択したデバイス"

#: pyanaconda/ui/gui/spokes/filter.glade:1434
msgctxt "GUI|Installation Destination|Filter"
msgid "_Add ZFCP LUN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1449
msgid "Add EC_KD DASD..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1464
msgctxt "GUI|Installation Destination|Filter"
msgid "Add _iSCSI Target..."
msgstr ""

#: pyanaconda/ui/gui/spokes/filter.glade:1479
msgctxt "GUI|Installation Destination|Filter"
msgid "Add FCo_E SAN..."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:16
msgid "bond"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:20
msgid "team"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:24
msgid "vlan"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:31
#, fuzzy
msgid "Add device"
msgstr "RAID デバイス"

#: pyanaconda/ui/gui/spokes/network.glade:45
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/network.glade:59
#, fuzzy
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Add"
msgstr "追加(_A)"

#: pyanaconda/ui/gui/spokes/network.glade:88
msgctxt "GUI|Network|Add Device Dialog"
msgid "_Select the type of device you wish to add"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:161
#: pyanaconda/ui/gui/spokes/network.glade:2273
msgid "NETWORK & HOSTNAME"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:200
msgid ""
"Please use the live desktop environment's tools for customizing your network "
"configuration.  You can set the hostname here."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:414
msgid "Slaves"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:438
#: pyanaconda/ui/gui/spokes/network.glade:874
#: pyanaconda/ui/gui/spokes/network.glade:1442
msgid "DNS"
msgstr "DNS"

#: pyanaconda/ui/gui/spokes/network.glade:464
#: pyanaconda/ui/gui/spokes/network.glade:861
#: pyanaconda/ui/gui/spokes/network.glade:1429
msgid "Default Route"
msgstr "デフォルトルート"

#: pyanaconda/ui/gui/spokes/network.glade:488
msgid "Subnet Mask"
msgstr "サブネットマスク"

#: pyanaconda/ui/gui/spokes/network.glade:560
#: pyanaconda/ui/gui/spokes/network.glade:825
#: pyanaconda/ui/gui/spokes/network.glade:1273
msgid "Speed"
msgstr "速度"

#: pyanaconda/ui/gui/spokes/network.glade:584
#: pyanaconda/ui/gui/spokes/network.glade:813
msgid "Hardware Address"
msgstr "ハードウェアアドレス"

#: pyanaconda/ui/gui/spokes/network.glade:608
msgid "Parent"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:620
msgid "Vlan ID"
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:695
#, fuzzy
msgctxt "GUI|Network|Wired"
msgid "C_onfigure..."
msgstr "設定する(_O) ..."

#: pyanaconda/ui/gui/spokes/network.glade:699
msgid "To apply the configuration immediately turn the device off and on."
msgstr ""

#: pyanaconda/ui/gui/spokes/network.glade:986
msgid "Security"
msgstr "セキュリティ"

#: pyanaconda/ui/gui/spokes/network.glade:998
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Network Name"
msgstr "ネットワーク名(_N)"

#: pyanaconda/ui/gui/spokes/network.glade:1012
msgid "Network Name"
msgstr "ネットワーク名"

#: pyanaconda/ui/gui/spokes/network.glade:1024
msgid "Security Key"
msgstr "セキュリティキー"

#: pyanaconda/ui/gui/spokes/network.glade:1108
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Use as Hotspot..."
msgstr "ホットスポットとして使用する(_U)..."

#: pyanaconda/ui/gui/spokes/network.glade:1125
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Stop Hotspot..."
msgstr "ホットスポットを停止する(_S)..."

#: pyanaconda/ui/gui/spokes/network.glade:1142
#, fuzzy
msgctxt "GUI|Network|Wireless"
msgid "_Configure..."
msgstr "設定する(_C)..."

#: pyanaconda/ui/gui/spokes/network.glade:1261
msgid "IMEI"
msgstr "IMEI"

#: pyanaconda/ui/gui/spokes/network.glade:1285
msgid "Provider"
msgstr "プロバイダー"

#: pyanaconda/ui/gui/spokes/network.glade:1467
#, fuzzy
msgctxt "GUI|Network|Mobile Broadband"
msgid "_Configure..."
msgstr "設定する(_C)..."

#: pyanaconda/ui/gui/spokes/network.glade:1578
msgid "VPN Type"
msgstr "VPN タイプ"

#: pyanaconda/ui/gui/spokes/network.glade:1590
msgid "Gateway"
msgstr "ゲートウェイ"

#: pyanaconda/ui/gui/spokes/network.glade:1602
msgid "Group Name"
msgstr "グループ名"

#: pyanaconda/ui/gui/spokes/network.glade:1614
msgid "Group Password"
msgstr "グループパスワード"

#: pyanaconda/ui/gui/spokes/network.glade:1626
#: pyanaconda/ui/gui/spokes/user.glade:118
msgid "Username"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/network.glade:1733
#, fuzzy
msgctxt "GUI|Network|VPN"
msgid "_Configure..."
msgstr "設定する(_C)..."

#: pyanaconda/ui/gui/spokes/network.glade:1856
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Method"
msgstr "メソッド(_M)"

#: pyanaconda/ui/gui/spokes/network.glade:1881
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Configuration URL"
msgstr "設定の URL(_C)"

#: pyanaconda/ui/gui/spokes/network.glade:1907
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "HTTP _Proxy"
msgstr "HTTP プロキシ(_H)"

#: pyanaconda/ui/gui/spokes/network.glade:1921
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "H_TTPS Proxy"
msgstr "HTTPS プロキシ(_T)"

#: pyanaconda/ui/gui/spokes/network.glade:1935
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_FTP Proxy"
msgstr "FTP プロキシ(_F)"

#: pyanaconda/ui/gui/spokes/network.glade:1949
#, fuzzy
msgctxt "GUI|Network|Proxy"
msgid "_Socks Host"
msgstr "Socks ホスト(_S)"

#: pyanaconda/ui/gui/spokes/network.glade:2142
#, fuzzy
msgid "Network Config Box"
msgstr "ネットワーク構成を書き込み中"

#: pyanaconda/ui/gui/spokes/network.glade:2158
#, fuzzy
msgctxt "GUI|Network"
msgid "Unloc_k"
msgstr "ロックを解除する"

#: pyanaconda/ui/gui/spokes/network.glade:2179
#, fuzzy
msgctxt "GUI|Network"
msgid "_Airplane Mode"
msgstr "航空機内モード(_P)"

#: pyanaconda/ui/gui/spokes/network.glade:2198
#, fuzzy
msgid "More Network Config Box"
msgstr "ネットワーク構成を書き込み中"

#: pyanaconda/ui/gui/spokes/network.glade:2218
#, fuzzy
msgctxt "GUI|Network"
msgid "_Hostname:"
msgstr "ホスト名:"

#: pyanaconda/ui/gui/spokes/network.glade:2279
msgid "NETWORK CONFIGURATION"
msgstr "ネットワーク設定"

#: pyanaconda/ui/gui/spokes/network.glade:2324
msgid ""
"We'll need network access to fetch information about your location and to "
"make software\n"
"updates available for you."
msgstr ""
"ユーザーの位置情報の取得およびソフトウェア更新を取得可能にするためネットワー"
"クアクセスが必要です。"

#: pyanaconda/ui/gui/spokes/network.glade:2354
#, fuzzy
msgid "Authentication"
msgstr "認証の使用(_U)"

#: pyanaconda/ui/gui/spokes/network.glade:2368
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/network.glade:2383
#, fuzzy
msgctxt "GUI|Network|Authentication Dialog"
msgid "C_onnect"
msgstr "接続済みです"

#: pyanaconda/ui/gui/spokes/network.glade:2457
msgid ""
"Passwords or encryption keys are required to access\n"
"the wireless network"
msgstr ""

#: pyanaconda/ui/gui/spokes/software.glade:10
msgid "SOFTWARE SELECTION"
msgstr "ソフトウェアの選択"

#: pyanaconda/ui/gui/spokes/software.glade:66
#, fuzzy
msgid "Base Environment"
msgstr "使用環境を選択します"

#: pyanaconda/ui/gui/spokes/software.glade:84
msgid "Add-Ons for Selected Environment"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:27
#, fuzzy
msgctxt "GUI|Storage|Need Space Dialog"
msgid "Cancel & _add more disks"
msgstr "取り消して、もっとディスクを追加(_A)"

#: pyanaconda/ui/gui/spokes/storage.glade:43
#, fuzzy
msgctxt "GUI|Storage|Need Space Dialog"
msgid "_Reclaim space"
msgstr "領域を確保する(_R)"

#: pyanaconda/ui/gui/spokes/storage.glade:78
#: pyanaconda/ui/gui/spokes/storage.glade:295
msgid "INSTALLATION OPTIONS"
msgstr "インストールオプション"

#: pyanaconda/ui/gui/spokes/storage.glade:95
#: pyanaconda/ui/gui/spokes/storage.glade:311
msgid ""
"Here we'll describe how much space is needed for the current software "
"selection."
msgstr "ここで、現在のソフトウェア選択に必要となる領域を説明します。"

#: pyanaconda/ui/gui/spokes/storage.glade:121
#: pyanaconda/ui/gui/spokes/storage.glade:337
msgid "disk free"
msgstr "disk free"

#: pyanaconda/ui/gui/spokes/storage.glade:137
#: pyanaconda/ui/gui/spokes/storage.glade:352
msgid "Free space available for use."
msgstr "使用できる空き領域"

#: pyanaconda/ui/gui/spokes/storage.glade:153
#: pyanaconda/ui/gui/spokes/storage.glade:368
msgid "fs free"
msgstr "fs free"

#: pyanaconda/ui/gui/spokes/storage.glade:169
#: pyanaconda/ui/gui/spokes/storage.glade:384
msgid "Free space unavailable but reclaimable from existing partitions."
msgstr ""
"現在利用できないものの、既存のパーティションから取り戻すことができる空き領域"

#: pyanaconda/ui/gui/spokes/storage.glade:192
#: pyanaconda/ui/gui/spokes/storage.glade:407
msgid "Here we'll describe what your options are."
msgstr "ここで、ユーザーのオプションを説明します。"

#: pyanaconda/ui/gui/spokes/storage.glade:219
#, fuzzy
msgid "Need Space"
msgstr "空き領域を使用する"

#: pyanaconda/ui/gui/spokes/storage.glade:244
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Quit installer"
msgstr "インストーラーを終了(_Q)"

#: pyanaconda/ui/gui/spokes/storage.glade:260
#, fuzzy
msgctxt "GUI|Storage|No Space Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/storage.glade:434
msgid "No Space"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:509
#, fuzzy
msgid "Device Selection"
msgstr "デバイスの説明"

#: pyanaconda/ui/gui/spokes/storage.glade:528
msgid ""
"Select the device(s) you'd like to install to.  They will be left untouched "
"until you click on the main menu's \"Begin Installation\" button."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:546
msgid "Local Standard Disks"
msgstr "ローカルの標準ディスク"

#: pyanaconda/ui/gui/spokes/storage.glade:610
#: pyanaconda/ui/gui/spokes/storage.glade:750
msgid "Disks left unselected here will not be touched."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:629
msgid "Specialized & Network Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:699
msgctxt "GUI|Storage"
msgid "_Add a disk..."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:716
msgid "Add Specialized Disk"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:769
msgid "Other Storage Options"
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:795
#, fuzzy
msgid "Partitioning"
msgstr "パーティション設定の終了(_F)"

#: pyanaconda/ui/gui/spokes/storage.glade:812
#, fuzzy
msgid "Encryption"
msgstr "暗号化する(_E)"

#: pyanaconda/ui/gui/spokes/storage.glade:825
#, fuzzy
msgid "A_utomatically configure partitioning."
msgstr "自動パーティション設定が選択されました"

#: pyanaconda/ui/gui/spokes/storage.glade:841
#, fuzzy
msgid "_I will configure partitioning."
msgstr "パーティション設定の終了(_F)"

#: pyanaconda/ui/gui/spokes/storage.glade:873
#, fuzzy
msgid "_Encrypt my data."
msgstr "暗号化する(_E)"

#: pyanaconda/ui/gui/spokes/storage.glade:888
#, fuzzy
msgid "You'll set a passphrase next."
msgstr "データを暗号化する。パスフレーズは後で設定。"

#: pyanaconda/ui/gui/spokes/storage.glade:909
msgid "I would like to _make additional space available."
msgstr ""

#: pyanaconda/ui/gui/spokes/storage.glade:925
#, fuzzy
msgid "Storage Options"
msgstr "自動パーティション設定のオプション"

#: pyanaconda/ui/gui/spokes/storage.glade:957
msgid "summary"
msgstr "要約"

#: pyanaconda/ui/gui/spokes/storage.glade:990
#, fuzzy
msgctxt "GUI|Storage"
msgid "_Full disk summary and bootloader..."
msgstr "完全なディスク要約とオプション... "

#: pyanaconda/ui/gui/spokes/keyboard.glade:15
#: pyanaconda/ui/gui/spokes/keyboard.glade:389
msgid "KEYBOARD LAYOUT"
msgstr "キーボードのレイアウト"

#: pyanaconda/ui/gui/spokes/keyboard.glade:57
msgid ""
"Which keyboard layouts would you like to use on this system?  You may move "
"any layout to the top of the list to select it as the default."
msgstr ""
"このシステムで使用するキーボードのレイアウトを選択してください。 いずれのレイ"
"アウトでも一覧の先頭に移動させデフォルトとして選択することができます。"

#: pyanaconda/ui/gui/spokes/keyboard.glade:96
msgid ""
"Changes here will only apply to the installed system. Use the desktop's tool "
"to configure the keyboard for the installation process."
msgstr ""
"ここでの変更は、インストールされた後のシステムに対して適用されます。インス"
"トールプロセスのためにキーボードレイアウトを設定するには、デスクトップのツー"
"ルを使用してください。"

#: pyanaconda/ui/gui/spokes/keyboard.glade:153
msgid "name"
msgstr "名前"

#: pyanaconda/ui/gui/spokes/keyboard.glade:167
#, fuzzy
msgid "Selected Layouts"
msgstr "選択したレイアウトを上に移動"

#: pyanaconda/ui/gui/spokes/keyboard.glade:190
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Add layout"
msgstr "レイアウトの追加"

#: pyanaconda/ui/gui/spokes/keyboard.glade:205
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Remove layout"
msgstr "レイアウトの削除"

#: pyanaconda/ui/gui/spokes/keyboard.glade:220
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _up"
msgstr "選択したレイアウトを上に移動"

#: pyanaconda/ui/gui/spokes/keyboard.glade:235
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "Move selected layout _down"
msgstr "選択したレイアウトを下に移動"

#: pyanaconda/ui/gui/spokes/keyboard.glade:251
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Preview layout"
msgstr "レイアウトのプレビュー"

#: pyanaconda/ui/gui/spokes/keyboard.glade:291
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Test the layout configuration below:"
msgstr "レイアウト設定のテストは出来ません。"

#: pyanaconda/ui/gui/spokes/keyboard.glade:332
msgid "Alt + Shift to switch layouts."
msgstr "Alt + Shift を使ってレイアウトを切り替えます"

#: pyanaconda/ui/gui/spokes/keyboard.glade:347
#, fuzzy
msgctxt "GUI|Keyboard Layout"
msgid "_Options"
msgstr "オプション(_O)"

#: pyanaconda/ui/gui/spokes/keyboard.glade:422
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/keyboard.glade:436
#, fuzzy
msgctxt "GUI|Keyboard Layout|Add Layout"
msgid "_Add"
msgstr "追加(_A)"

#: pyanaconda/ui/gui/spokes/keyboard.glade:468
msgid "ADD A KEYBOARD LAYOUT"
msgstr "キーボードレイアウトを追加"

#: pyanaconda/ui/gui/spokes/keyboard.glade:485
msgid "You may add a keyboard layout by selecting it below:"
msgstr "以下の表で選択してキーボードレイアウトを追加することができます"

#: pyanaconda/ui/gui/spokes/keyboard.glade:521
#: pyanaconda/ui/gui/spokes/source.glade:991
#: pyanaconda/ui/gui/spokes/lib/cart.glade:133
#: pyanaconda/ui/gui/spokes/lib/resize.glade:163
msgid "Name"
msgstr "名前"

#: pyanaconda/ui/gui/spokes/keyboard.glade:534
#, fuzzy
msgid "Available Layouts"
msgstr "使用できる地域"

#: pyanaconda/ui/gui/spokes/keyboard.glade:576
#, fuzzy
msgid "Add Layout"
msgstr "レイアウトの追加"

#: pyanaconda/ui/gui/spokes/keyboard.glade:608
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/keyboard.glade:622
#, fuzzy
msgctxt "GUI|Keyboard Layout|Switching Options"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/keyboard.glade:654
msgid "LAYOUT SWITCHING OPTIONS"
msgstr "レイアウト切り替えのオプション"

#: pyanaconda/ui/gui/spokes/keyboard.glade:671
msgid ""
"Which combination(s) would you prefer for switching between keyboard layouts?"
msgstr "切り替えを行うキーボードレイアウトの組み合わせを選択してください。"

#: pyanaconda/ui/gui/spokes/keyboard.glade:707
msgid "use"
msgstr "使用"

#: pyanaconda/ui/gui/spokes/keyboard.glade:720
msgid "description"
msgstr "詳細"

#: pyanaconda/ui/gui/spokes/keyboard.glade:754
#, fuzzy
msgid "Layout Options"
msgstr "NFS マウントオプション:"

#: pyanaconda/ui/gui/spokes/langsupport.glade:33
#, fuzzy
msgid "LANGUAGE SUPPORT"
msgstr "言語"

#: pyanaconda/ui/gui/spokes/langsupport.glade:79
msgid "Select additional language support to be installed:"
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:138
#: pyanaconda/ui/gui/spokes/langsupport.glade:242
#: pyanaconda/ui/gui/spokes/welcome.glade:273
#: pyanaconda/ui/gui/spokes/welcome.glade:373
msgid "nativeName"
msgstr "nativeName"

#: pyanaconda/ui/gui/spokes/langsupport.glade:155
#: pyanaconda/ui/gui/spokes/welcome.glade:290
msgid "englishName"
msgstr "englishName"

#: pyanaconda/ui/gui/spokes/langsupport.glade:173
#: pyanaconda/ui/gui/spokes/welcome.glade:308
#, fuzzy
msgid "selected"
msgstr "何も選択されていません"

#: pyanaconda/ui/gui/spokes/langsupport.glade:194
#: pyanaconda/ui/gui/spokes/welcome.glade:337
msgid "Type here to search."
msgstr ""

#: pyanaconda/ui/gui/spokes/langsupport.glade:228
msgid "checked"
msgstr ""

#: pyanaconda/ui/gui/spokes/password.glade:8
#: pyanaconda/ui/gui/spokes/password.glade:203
msgid "ROOT PASSWORD"
msgstr "root パスワード"

#: pyanaconda/ui/gui/spokes/password.glade:54
#, fuzzy
msgctxt "GUI|Password"
msgid "_Root Password:"
msgstr "root パスワード:"

#: pyanaconda/ui/gui/spokes/password.glade:68
#, fuzzy
msgctxt "GUI|Password"
msgid "_Confirm:"
msgstr "確認:"

#: pyanaconda/ui/gui/spokes/password.glade:104
#: pyanaconda/ui/gui/spokes/user.glade:189
#, fuzzy
msgid "Confirm Password"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/password.glade:150
#: pyanaconda/ui/gui/spokes/user.glade:252
#, fuzzy
msgid "empty password"
msgstr "root パスワードを設定する"

#: pyanaconda/ui/gui/spokes/password.glade:171
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"システムの管理には root アカウントを使用します。 root ユーザーのパスワードを"
"入力してください。"

#: pyanaconda/ui/gui/spokes/source.glade:29
#, fuzzy
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/source.glade:43
msgctxt "GUI|Software Source|ISO Chooser Dialog"
msgid "_Open"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:95
#, fuzzy
msgctxt "GUI|Software Source|Media Check Dialog"
msgid "_Done"
msgstr "完了(_D)"

#: pyanaconda/ui/gui/spokes/source.glade:128
msgid "MEDIA VERIFICATION"
msgstr "メディアの確認"

#: pyanaconda/ui/gui/spokes/source.glade:155
msgid "Verifying media, please wait..."
msgstr "メディアを確認しています。お待ち下さい..."

#: pyanaconda/ui/gui/spokes/source.glade:224
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/source.glade:239
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Ok"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/source.glade:267
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Enable HTTP Proxy"
msgstr "HTTP プロキシを有効にする(_E)"

#: pyanaconda/ui/gui/spokes/source.glade:300
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Proxy URL"
msgstr "<b>プロキシの URL</b>"

#: pyanaconda/ui/gui/spokes/source.glade:317
msgid "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"
msgstr "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/source.glade:349
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "_Use Authentication"
msgstr "認証の使用(_U)"

#: pyanaconda/ui/gui/spokes/source.glade:376
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "User_name"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/source.glade:393
#, fuzzy
msgctxt "GUI|Software Source|Proxy Dialog"
msgid "Pass_word"
msgstr "パスワード: "

#: pyanaconda/ui/gui/spokes/source.glade:473
msgid "INSTALLATION SOURCE"
msgstr "インストールソース"

#: pyanaconda/ui/gui/spokes/source.glade:532
msgid "Which installation source would you like to use?"
msgstr "使用するインストールソースを選択してください。"

#: pyanaconda/ui/gui/spokes/source.glade:546
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Auto-detected installation media:"
msgstr "自動検出したインストールメディア(_A):"

#: pyanaconda/ui/gui/spokes/source.glade:573
msgid "Device:"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/source.glade:585
#, fuzzy
msgid "Label:"
msgstr "<b>ラベル:</b>"

#: pyanaconda/ui/gui/spokes/source.glade:594
#: pyanaconda/ui/gui/spokes/source.glade:694
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Verify"
msgstr "確認する(_V)"

#: pyanaconda/ui/gui/spokes/source.glade:618
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_ISO file:"
msgstr "ISO ファイル(_I):"

#: pyanaconda/ui/gui/spokes/source.glade:646
#, fuzzy
msgctxt "GUI|Software Source"
msgid "D_evice:"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/source.glade:677
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Choose an ISO"
msgstr "ISO を1つ選択(_C)"

#: pyanaconda/ui/gui/spokes/source.glade:718
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_On the network:"
msgstr "ネットワーク上(_N):"

#: pyanaconda/ui/gui/spokes/source.glade:782
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Proxy setup..."
msgstr "プロキシの設定(_P)..."

#: pyanaconda/ui/gui/spokes/source.glade:797
#, fuzzy
msgctxt "GUI|Software Source"
msgid "This URL refers to a _mirror list."
msgstr "この URL はミラーリストを参照します。"

#: pyanaconda/ui/gui/spokes/source.glade:835
#, fuzzy
msgctxt "GUI|Software Source"
msgid "N_FS mount options:"
msgstr "NFS マウントオプション:"

#: pyanaconda/ui/gui/spokes/source.glade:850
#: pyanaconda/ui/gui/spokes/source.glade:851
msgid "This field is optional."
msgstr "このフィールドはオプションです。"

#: pyanaconda/ui/gui/spokes/source.glade:885
msgid "Updates"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:912
msgctxt "GUI|Software Source"
msgid ""
"Don't install the latest available software _updates. Install the default "
"versions provided by the installation source above."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:937
#, fuzzy
msgid "Additional repositories"
msgstr "Fedora Peopleのレポジトリ"

#: pyanaconda/ui/gui/spokes/source.glade:978
msgid "Enabled"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1023
#: pyanaconda/ui/gui/spokes/source.glade:1024
#, fuzzy
msgid "Add a new repository."
msgstr "新しいマウントポイントを追加します。"

#: pyanaconda/ui/gui/spokes/source.glade:1025
msgctxt "GUI|Software Source"
msgid "A_dd"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1042
#: pyanaconda/ui/gui/spokes/source.glade:1043
#, fuzzy
msgid "Remove the selected repository."
msgstr "選択したマウントポイントを削除します。"

#: pyanaconda/ui/gui/spokes/source.glade:1044
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Remove"
msgstr "削除(_R)"

#: pyanaconda/ui/gui/spokes/source.glade:1061
#: pyanaconda/ui/gui/spokes/source.glade:1062
msgid "Revert to the previous list of repositories."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1063
msgctxt "GUI|Software Source"
msgid "Rese_t"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1092
#: pyanaconda/ui/gui/spokes/source.glade:1093
msgid "URL for the repository, without protocol."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1106
#: pyanaconda/ui/gui/spokes/source.glade:1107
msgid "URL of proxy in the form of protocol://host:[port]"
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1120
#: pyanaconda/ui/gui/spokes/source.glade:1121
#, fuzzy
msgid "Optional proxy username."
msgstr "操作上のエラー"

#: pyanaconda/ui/gui/spokes/source.glade:1134
#, fuzzy
msgid "Optional proxy password."
msgstr "root パスワードを設定する"

#: pyanaconda/ui/gui/spokes/source.glade:1146
#, fuzzy
msgctxt "GUI|Software Source"
msgid "This URL refers to a mirror _list."
msgstr "この URL はミラーリストを参照します。"

#: pyanaconda/ui/gui/spokes/source.glade:1166
#, fuzzy
msgctxt "GUI|Software Source"
msgid "_Name:"
msgstr "名前:"

#: pyanaconda/ui/gui/spokes/source.glade:1182
#: pyanaconda/ui/gui/spokes/source.glade:1183
msgid "Protocol for the repository URL."
msgstr ""

#: pyanaconda/ui/gui/spokes/source.glade:1204
#, fuzzy
msgctxt "GUI|Software Source"
msgid "Pro_xy URL:"
msgstr "<b>プロキシの URL</b>"

#: pyanaconda/ui/gui/spokes/source.glade:1221
#, fuzzy
msgctxt "GUI|Software Source"
msgid "U_sername:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/source.glade:1238
#, fuzzy
msgctxt "GUI|Software Source"
msgid "Pass_word:"
msgstr "パスワード: "

#: pyanaconda/ui/gui/spokes/source.glade:1254
#: pyanaconda/ui/gui/spokes/source.glade:1255
msgid "Name of the repository."
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:23
#, fuzzy
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_Get me out of here!"
msgstr "ここから脱出したい! (_G)"

#: pyanaconda/ui/gui/spokes/welcome.glade:37
#, fuzzy
msgctxt "GUI|Welcome|Beta Warn Dialog"
msgid "_I accept my fate."
msgstr "運命を受け入れます(_I)"

#: pyanaconda/ui/gui/spokes/welcome.glade:68
msgid "This is unstable, pre-release software."
msgstr "これは不安定なプレリリースのソフトウェアです。"

#: pyanaconda/ui/gui/spokes/welcome.glade:97
msgid ""
"You wake up inside an OS installer in Timbuktu, and it's six months in the "
"future.  But, there are bugs.  Bugs everywhere.  Bugs you must live with.  "
"This OS of the future isn't a stable OS you can rely on.  It's for testing "
"purposes only."
msgstr ""
"Timbuktu の OS インストーラー内で目覚めました。そして 6 か月だけ未来です。し"
"かし、バグがあります。バグだらけです。共存すべきバグです。未来のこの OS は信"
"頼できるほど安定していません。これはテスト目的のみのものです。"

#: pyanaconda/ui/gui/spokes/welcome.glade:126
msgid "Beta Warn"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:225
msgid "What language would you like to use during the installation process?"
msgstr "インストール時に使用する言語を選択してください。"

#: pyanaconda/ui/gui/spokes/welcome.glade:316
msgid "Languages"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:389
msgid "Locales"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:423
msgid "WELCOME"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:444
#, fuzzy
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/spokes/welcome.glade:458
#, fuzzy
msgctxt "GUI|Welcome|Unsupported Hardware Dialog"
msgid "_Continue"
msgstr "続行(_C)"

#: pyanaconda/ui/gui/spokes/welcome.glade:489
msgid "Unsupported Hardware Detected"
msgstr ""

#: pyanaconda/ui/gui/spokes/welcome.glade:547
msgid "Unsupported Hardware"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:10
#: pyanaconda/ui/gui/spokes/user.glade:346
msgid "CREATE USER"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:62
msgctxt "GUI|User"
msgid "_Full name"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:80
#, fuzzy
msgctxt "GUI|User"
msgid "_Username"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/user.glade:101
#, fuzzy
msgid "Full Name"
msgstr "グループ名"

#: pyanaconda/ui/gui/spokes/user.glade:133
#, fuzzy
msgctxt "GUI|User"
msgid "_Password"
msgstr "パスワード: "

#: pyanaconda/ui/gui/spokes/user.glade:151
#, fuzzy
msgctxt "GUI|User"
msgid "_Confirm password"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/user.glade:203
msgid ""
"<b>Tip:</b> Keep your username shorter than 32 characters and do not use "
"spaces."
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:213
msgctxt "GUI|User"
msgid "Require a password to use this account"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:271
msgctxt "GUI|User"
msgid "Make this user administrator"
msgstr ""

#: pyanaconda/ui/gui/spokes/user.glade:289
msgctxt "GUI|User"
msgid "_Advanced..."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:27
msgid "Btrfs"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:62
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:540
msgid "RAID0 <span foreground=\"grey\">(Performance)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:66
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:544
msgid "RAID1 <span foreground=\"grey\">(Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:70
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:548
msgid "RAID4 <span foreground=\"grey\">(Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:74
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:552
msgid "RAID5 <span foreground=\"grey\">(Distributed Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:78
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:556
msgid "RAID6 <span foreground=\"grey\">(Redundant Error Checking)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:82
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:560
msgid "RAID10 <span foreground=\"grey\">(Performance, Redundancy)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:184
#, fuzzy
msgid "Add a new mount point."
msgstr "新しいマウントポイントを追加します。"

#: pyanaconda/ui/gui/spokes/custom.glade:185
#, fuzzy
msgctxt "GUI|Custom Partitioning"
msgid "Add"
msgstr "追加(_A)"

#: pyanaconda/ui/gui/spokes/custom.glade:200
#, fuzzy
msgid "Remove the selected mount point(s)."
msgstr "選択したマウントポイントを削除します。"

#: pyanaconda/ui/gui/spokes/custom.glade:201
#, fuzzy
msgctxt "GUI|Custom Partitioning"
msgid "Remove"
msgstr "削除(_R)"

#: pyanaconda/ui/gui/spokes/custom.glade:216
#, fuzzy
msgid "Reload storage configuration from disk."
msgstr "ストレージの設定に失敗しました: %s"

#: pyanaconda/ui/gui/spokes/custom.glade:217
msgctxt "GUI|Custom Partitioning"
msgid "Refresh"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:282
#: pyanaconda/ui/gui/spokes/custom.glade:1014
#: pyanaconda/ui/gui/spokes/custom.glade:1158
#: pyanaconda/ui/gui/spokes/custom.glade:1245
msgid "Selected Device"
msgstr "選択したデバイス"

#: pyanaconda/ui/gui/spokes/custom.glade:300
#: pyanaconda/ui/gui/spokes/custom.glade:1032
#: pyanaconda/ui/gui/spokes/custom.glade:1176
#: pyanaconda/ui/gui/spokes/custom.glade:1263
msgid "Device description"
msgstr "デバイスの説明"

#: pyanaconda/ui/gui/spokes/custom.glade:361
#, fuzzy
msgid "Mount _Point:"
msgstr "<b>マウントポイント:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:409
#, fuzzy
msgid "_Desired Capacity:"
msgstr "<b>要望する容量:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:461
#, fuzzy
msgid "_Label:"
msgstr "<b>ラベル:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:509
#, fuzzy
msgid "Device(s):"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/custom.glade:539
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure|Devices"
msgid "_Modify..."
msgstr "変更 ..."

#: pyanaconda/ui/gui/spokes/custom.glade:571
#, fuzzy
msgid "_Name:"
msgstr "名前:"

#: pyanaconda/ui/gui/spokes/custom.glade:627
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Device _Type:"
msgstr "<b>デバイスタイプ:</b>"

#: pyanaconda/ui/gui/spokes/custom.glade:669
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Encrypt"
msgstr "暗号化する(_E)"

#: pyanaconda/ui/gui/spokes/custom.glade:710
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "File S_ystem:"
msgstr "ファイルシステム"

#: pyanaconda/ui/gui/spokes/custom.glade:745
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "Ref_ormat"
msgstr "再フォーマット"

#: pyanaconda/ui/gui/spokes/custom.glade:799
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Volume Group:"
msgstr "ボリュームグループ:"

#: pyanaconda/ui/gui/spokes/custom.glade:851
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "_Modify..."
msgstr "変更 ..."

#: pyanaconda/ui/gui/spokes/custom.glade:883
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure"
msgid "RA_ID Level:"
msgstr "RAID デバイス"

#: pyanaconda/ui/gui/spokes/custom.glade:946
msgid "_Update Settings"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:969
msgid ""
"Note:  The settings you make on this screen will not be applied until you "
"click on the main menu's 'Begin Installation' button."
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1066
msgid ""
"This device is encrypted and cannot be read without a valid passphrase. You "
"may unlock it below."
msgstr ""
"このデバイスは暗号化されているため有効なパスフレーズがないと読み込めません。"
"以下でアンロックできます。"

#: pyanaconda/ui/gui/spokes/custom.glade:1087
#, fuzzy
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Passphrase:"
msgstr "パスフレーズ:"

#: pyanaconda/ui/gui/spokes/custom.glade:1115
#, fuzzy
msgctxt "GUI|Custom Partitioning|Encrypted"
msgid "_Unlock"
msgstr "アンロック(_U)"

#: pyanaconda/ui/gui/spokes/custom.glade:1210
msgid ""
"This device cannot be edited directly. You can remove it or select a "
"different device."
msgstr ""
"このデバイスの直接編集はできません。 デバイスを削除するか別のデバイスを選択し"
"ます。"

#: pyanaconda/ui/gui/spokes/custom.glade:1358
msgid "AVAILABLE SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1411
msgid "TOTAL SPACE"
msgstr ""

#: pyanaconda/ui/gui/spokes/custom.glade:1451
msgctxt "GUI|Custom Partitioning"
msgid "_Reset All"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:23
#, fuzzy
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:56
msgid ""
"Please select the network interface which is connected to your FCoE switch."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:70
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_NIC:"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:92
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use _DCB"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:108
msgctxt "GUI|Advanced Storage|FCoE"
msgid "Use auto _vlan"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:144
msgctxt "GUI|Advanced Storage|FCoE"
msgid "_Add FCoE Disk(s)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:187
msgid "Error adding FCoE SAN."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/fcoe.glade:215
msgid ""
"The following error occurred while attempting to add an FCoE SAN.  Please "
"check your configuration and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:38
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:52
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:84
msgid "ADD iSCSI STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:113
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:141
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Target IP Address:"
msgstr "IP アドレス"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:169
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "iSCSI _Initiator Name:"
msgstr "iSCSI イニシエーター名を入力しなければなりません"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:185
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:743
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:186
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:744
msgid "CHAP pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:187
#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:745
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:202
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Discovery Authentication Type:"
msgstr "認証の使用(_U)"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:217
#, fuzzy
msgid ""
"<span size=\"small\"><b>Example:</b> iqn.2012-09.com.example:diskarrays-sn-"
"a8675309</span>"
msgstr "<span size=\"small\"><b>Example:</b> squid.mysite.org:3128</span>"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:262
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Username:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:276
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|CHAP"
msgid "CHAP _Password:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:385
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Username:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:399
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:413
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP User_name:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:427
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:460
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Start"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:497
msgid "Discovering iSCSI targets.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:537
msgid "Discovery login rejected."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:552
msgid ""
"The following error occurred discovering iSCSI targets.  Please double check "
"your authorization information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:574
msgctxt "GUI|Advanced Storage|iSCSI|Configure|Retry"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:614
msgctxt "GUI|Advanced Storage|iSCSI|Configure"
msgid "_Bind targets to network interfaces"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:689
#, fuzzy
msgid "Node Name"
msgstr "ネットワーク名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:700
msgid "Interface"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:729
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login"
msgid "_Node login authentication type:"
msgstr "認証の使用(_U)"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:746
msgid "Use the credentials from discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:781
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Username:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:795
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|CHAP"
msgid "CHAP _Password:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:904
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Username:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:918
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "CHAP _Password:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:932
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "_Reverse CHAP Username:"
msgstr "ユーザー名"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:946
#, fuzzy
msgctxt "GUI|Advanced Storage|iSCSI|Login|Reverse CHAP"
msgid "Reverse CHAP Pass_word:"
msgstr "VNC パスワード"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:994
msgctxt "GUI|Advanced Storage|iSCSI|Login|Login"
msgid "_Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1050
#, fuzzy
msgid "Node login failed."
msgstr "接続に失敗しました"

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1065
msgid ""
"The following error occurred logging into the selected iSCSI node.  Please "
"double check your authorization information and try again"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/iscsi.glade:1087
msgctxt "GUI|Advanced Storage|iSCSI|Login|Retry"
msgid "Retry _Log In"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:22
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:36
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:68
msgid "ADD DASD STORAGE TARGET"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:97
msgid "To use DASD disks, you must provide the device number."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:126
#, fuzzy
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Device number:"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:146
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Start Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:183
msgid "Discovering DASD devices.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:225
msgid "Device discovery failed."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:242
msgid ""
"The following error occurred discovering DASD devices.  Please double check "
"your configuration information and try again."
msgstr ""

#: pyanaconda/ui/gui/spokes/advstorage/dasd.glade:268
msgctxt "GUI|Advanced Storage|DASD|Configure"
msgid "_Retry Discovery"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/cart.glade:26
#: pyanaconda/ui/gui/spokes/lib/cart.glade:78
msgid "SELECTED DISKS"
msgstr "選択したディスク"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:43
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Close"
msgstr "閉じる(_C)"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:106
msgid "Boot"
msgstr "起動"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:120
msgid "Description"
msgstr "説明"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:158
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:248
#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:741
msgid "Free"
msgstr "空き"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:199
#, fuzzy
msgctxt "GUI|Selected Disks Dialog"
msgid "_Remove"
msgstr "削除(_R)"

#: pyanaconda/ui/gui/spokes/lib/cart.glade:226
msgid "Disk summary goes here"
msgstr "ディスクの要約はここへ"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:21
#, fuzzy
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:37
#, fuzzy
msgctxt "GUI|Custom Partitioning|Confirm Delete Dialog"
msgid "_Delete It"
msgstr "それを削除します(_D)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:129
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:144
#, fuzzy
msgctxt "GUI|Custom Partitioning|Configure Dialog"
msgid "_Select"
msgstr "選択(_S)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:176
msgid "CONFIGURE MOUNT POINT"
msgstr "マウントポイントの設定"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:191
msgid "Select one or more disks this device may reside on."
msgstr "このデバイスを存在させるディスクをひとつまたは複数選択してください。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:321
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:336
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Add mount point"
msgstr "マウントポイントの追加(_A)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:369
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "_Desired Capacity:"
msgstr "<b>要望する容量:</b>"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:386
#, fuzzy
msgctxt "GUI|Custom Partitioning|Add Dialog"
msgid "Mount _Point:"
msgstr "<b>マウントポイント:</b>"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:404
msgid "ADD A NEW MOUNT POINT"
msgstr "新規のマウントポイントの追加"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:421
msgid ""
"More customization options are available\n"
"after creating the mount point below."
msgstr ""
"以下にマウントポイントを作成した後に、他のカスタム化オプションが利用できるよ"
"うになります。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:456
msgid ""
"That mount point is already in\n"
"use.  Try something else?"
msgstr ""
"そのマウントポイントはすでに使用中です。\n"
"　他を試しますか？"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:480
msgid "eg: \"20 GB\", \"500mb\" (minus the quotation marks)"
msgstr "例: \"20 GB\"、 \"500mb\" (引用符は除く)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:536
msgid "Single <span foreground=\"grey\">(No Redundancy, No Striping)</span>"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:584
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:598
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Save"
msgstr "保存(_S)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:630
#, fuzzy
msgid "CONFIGURE CONTAINER"
msgstr "マウントポイントの設定"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:645
#, fuzzy
msgid ""
"Please create a name for this container and select at least one disk below."
msgstr ""
"ボリュームグループ名を決めて作成し、以下から少なくても1つのディスクを選択して"
"ください。"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:663
#, fuzzy
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "_Name:"
msgstr "名前:"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:783
#, fuzzy
msgid "RAID Level:"
msgstr "RAID デバイス"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:811
#, fuzzy
msgid "Encrypt"
msgstr "暗号化する(_E)"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:852
#, fuzzy
msgid "Please enter a valid name."
msgstr "有効なマウントポイントを記入して下さい"

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:882
msgctxt "GUI|Custom Partitioning|Container Dialog"
msgid "Si_ze policy:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:897
msgid "Automatic"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:898
msgid "As large as possible"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.glade:899
msgid "Fixed"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:8
msgid "UNFORMATTED DASDS"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:23
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:37
#, fuzzy
msgctxt "GUI|DAD Format Dialog"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:70
msgid ""
"The following unformatted DASDs have been detected on your system. You can "
"choose to format them now with dasdfmt or cancel to leave them unformatted. "
"Unformatted DASDs can not be used during installation."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:86
#, fuzzy
msgid "Unformatted DASDs Detected"
msgstr "DASD デバイスをフォーマット中..."

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:136
msgid ""
"<b>Warning</b>: All storage changes made using the installer will be lost "
"when you choose to format."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:160
msgctxt "GUI|DAD Format Dialog"
msgid "_Format with dasdfmt"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:207
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while formatting completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:247
msgid "Disk formatting complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/dasdfmt.glade:262
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:269
msgid ""
"Pressing 'OK' below will take you to the disk selection screen where you "
"will need to re-select your disks."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/detailederror.glade:25
msgid "An unknown error occurred during installation.  Details are below."
msgstr "インストール中に不明なエラーが発生しました。詳細は以下にあります。"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:23
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:38
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Save Passphrase"
msgstr "パスフレーズの保存"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:66
msgid "DISK ENCRYPTION PASSPHRASE"
msgstr "ディスク暗号化用パスフレーズ"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:82
msgid ""
"You have chosen to encrypt some of your data. You will need to create a "
"passphrase that you will use to access your data when you start your "
"computer."
msgstr ""
"データの一部を暗号化する選択をしました。コンピュータの開始時にデータへのアク"
"セスに使用するパスフレーズを作成する必要があります。"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:103
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "_Passphrase:"
msgstr "パスフレーズ:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:132
#, fuzzy
msgctxt "GUI|Passphrase Dialog"
msgid "Con_firm:"
msgstr "確認:"

#: pyanaconda/ui/gui/spokes/lib/passphrase.glade:270
#, fuzzy
msgid ""
"Warning: You won't be able to switch between keyboard layouts (from the "
"default one) when you decrypt your disks after install."
msgstr ""
"警告: 現在のキーボートレイアウトは <b>%s</b> です。キーボードのレイアウトを変"
"更すると、インストールの後にディスクの復号が出来ないかもしれません。"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:8
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:71
#, fuzzy
msgid "RESCAN DISKS"
msgstr "ディスク領域の獲得"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:25
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:39
#, fuzzy
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_OK"
msgstr "OK"

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:87
msgid ""
"You can remove or insert additional disks at this time and press 'Rescan "
"Disks' below for the changes to take effect."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:118
msgid ""
"<b>Warning:</b> All storage changes made using the installer will be lost "
"when you press 'Rescan Disks'."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:143
msgctxt "GUI|Refresh Dialog|Rescan"
msgid "_Rescan Disks"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:176
#: pyanaconda/ui/gui/spokes/lib/refresh.glade:212
msgid "Scanning disks.  This may take a moment..."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:223
msgid ""
"You may <a href=\"\">go back to the main menu</a> to complete other\n"
"installation options while this scan completes."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/refresh.glade:254
msgid "Disk rescan complete."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/resize.glade:55
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Cancel"
msgstr "取り消し(_C)"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:70
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Reclaim space"
msgstr "領域を確保する(_R)"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:104
msgid "RECLAIM DISK SPACE"
msgstr "ディスク領域の獲得"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:121
msgid "Description goes here."
msgstr "説明"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:174
msgid "Filesystem"
msgstr "ファイルシステム"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:185
msgid "Reclaimable Space"
msgstr "獲得可能な領域"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:196
#: pyanaconda/ui/gui/spokes/lib/summary.glade:141
msgid "Action"
msgstr "アクション"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:221
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Preserve"
msgstr "保存"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:237
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Delete"
msgstr "削除"

#: pyanaconda/ui/gui/spokes/lib/resize.glade:253
#, fuzzy
msgctxt "GUI|Reclaim Dialog"
msgid "_Shrink"
msgstr "縮小"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:23
#: pyanaconda/ui/gui/spokes/lib/summary.glade:87
msgid "SUMMARY OF CHANGES"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:40
#, fuzzy
msgctxt "GUI|Summary Dialog"
msgid "_Cancel & Return to Custom Partitioning"
msgstr "カスタムパーティション設定が選択されました"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:55
#, fuzzy
msgctxt "GUI|Summary Dialog"
msgid "_Accept Changes"
msgstr "変更を適用(_A)"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:103
msgid ""
"Your customizations will result in the following changes taking effect after "
"you return to the main menu and begin installation:"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:130
msgid "Order"
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/summary.glade:163
#, fuzzy
msgid "Device Name"
msgstr "デバイス:"

#: pyanaconda/ui/gui/spokes/lib/summary.glade:174
#, fuzzy
msgid "Mountpoint"
msgstr "マウントポイントの追加(_A)"

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:35
msgid ""
"The system needs better quality of random data, you can improve it by typing "
"randomly on keyboard and moving your mouse. The installation will continue "
"automatically regardless of random data quality when time runs out."
msgstr ""

#: pyanaconda/ui/gui/spokes/lib/entropy_dialog.glade:53
msgid "Random data quality:"
msgstr ""

#: pyanaconda/ui/gui/main.glade:24
msgid "_Exit Installer"
msgstr "インストーラーを終了する(_E)"

#: pyanaconda/ui/gui/main.glade:94
#, fuzzy
msgctxt "GUI|Main"
msgid "_No"
msgstr "いいえ(_N)"

#: pyanaconda/ui/gui/main.glade:108
#, fuzzy
msgctxt "GUI|Main"
msgid "_Yes"
msgstr "はい(_Y)"

#: pyanaconda/ui/gui/main.glade:151
msgid ""
"Are you sure you wish to quit the\n"
"installation process?"
msgstr ""
"インストールプロセスを本当に\n"
"終了したいですか？"

#: pyanaconda/ui/gui/main.glade:175
#, fuzzy
msgid "Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/hubs/summary.glade:8
#: pyanaconda/ui/gui/hubs/summary.glade:143
msgid "INSTALLATION SUMMARY"
msgstr "インストール概要"

#: pyanaconda/ui/gui/hubs/summary.glade:80
#, fuzzy
msgctxt "GUI|Summary"
msgid "_Quit"
msgstr "終了する (_Q)"

#: pyanaconda/ui/gui/hubs/summary.glade:96
#, fuzzy
msgctxt "GUI|Summary"
msgid "_Begin Installation"
msgstr "インストールの開始(_B)"

#: pyanaconda/ui/gui/hubs/summary.glade:127
msgid "We won't touch your disks until you click 'Begin Installation'."
msgstr ""

#: pyanaconda/ui/gui/hubs/progress.glade:8
#: pyanaconda/ui/gui/hubs/progress.glade:257
msgid "CONFIGURATION"
msgstr "設定"

#: pyanaconda/ui/gui/hubs/progress.glade:87
msgid "Preparing to install"
msgstr "インストールの準備中"

#: pyanaconda/ui/gui/hubs/progress.glade:145
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Finish configuration"
msgstr "設定完了(_F)"

#: pyanaconda/ui/gui/hubs/progress.glade:198
#, fuzzy
msgctxt "GUI|Progress"
msgid "_Reboot"
msgstr "再起動(_R)"

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install to Hard Drive"
msgstr "ハードディスクにインストール"

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install"
msgstr "インストール"

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install the live CD to your hard disk"
msgstr "ライブ ディスク をハードディスクにインストール"

#: /home/vpodzime/anaconda/data/liveinst/console.apps/liveinst:5
msgid "Starting Install to Hard Drive"
msgstr "ハードディスクへのインストールを開始します"

#: data/liveinst/gnome/fedora-welcome.desktop.in.h:1
#: data/liveinst/gnome/fedora-welcome:65
msgid "Welcome to Fedora"
msgstr "Fedora へようこそ"

#: anaconda:594
#, fuzzy, python-format
msgid ""
"%(product_name)s requires %(needed_ram)s MB of memory to install, but you "
"only have %(total_ram)s MB on this machine.\n"
msgstr ""
"インストールするために %s は、最低 %s MB のメモリーを必要とします。お使いのシ"
"ステムには %s MB しか搭載されていません。\n"

#: anaconda:596
#, fuzzy, python-format
msgid ""
"The %(product_name)s graphical installer requires %(needed_ram)s MB of "
"memory, but you only have %(total_ram)s MB\n"
"."
msgstr ""
"%s のグラフィカルインストーラーは、最低 %s MB のメモリーを必要とします。お使"
"いのシステムには %s MB のメモリーしか搭載されていません。"

#: anaconda:599
msgid ""
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: anaconda:601
msgid "Not enough RAM"
msgstr "メモリーが足りません。"

#: anaconda:602
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""
"次のとおり実行してテキストモードインストーラーを試して下さい:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
"なお、この操作は root でのターミナルから実行します。"

#: anaconda:605
msgid " Starting text mode."
msgstr " テキストモードを開始しています。"

#: anaconda:642
#, fuzzy
msgid "Fatal Error"
msgstr "エラー"

#: anaconda:767
msgid "DISPLAY variable not set. Starting text mode."
msgstr "DISPLAY 変数がセットされていません。テキストモードで起動します。"

#: anaconda:775
msgid "Graphical installation is not available. Starting text mode."
msgstr "グラフィカルインストールは使用できません。テキストモードを起動します。"

#: anaconda:783
#, fuzzy
msgid ""
"Text mode provides a limited set of installation options. It does not offer "
"custom partitioning for full control over the disk layout. Would you like to "
"use VNC mode instead?"
msgstr ""
"テキストモードによるインストールは限定的なインストールオプションしか提供され"
"ず、パーティション設定やパッケージ選択の指定が行えません。代わりに VNC モード"
"を使用しますか？"

#: anaconda:872
#, python-format
msgid "Please ssh install@%s to begin the install."
msgstr "ssh install@%s を実行して、インストールを始めて下さい。"

#: anaconda:874
msgid "Please ssh install@<host> to continue installation."
msgstr "ssh install@<host> を実行して、インストールを継続して下さい。"

#: data/liveinst/gnome/fedora-welcome:94
msgid "Try Fedora"
msgstr "Fedora を試す"

#: data/liveinst/gnome/fedora-welcome:110
msgid ""
"You are currently running Fedora from live media.\n"
"You can install Fedora now, or choose \"Install to Hard Drive\" in the "
"Activities Overview at any later time."
msgstr ""
"現在、ライブメディアから Fedora を稼働しています。\n"
"今、Fedora をインストールできます。または後でアクティビティオーバービューで"
"「ハードドライブにインストールする」を選択することもできます。"

#: data/liveinst/gnome/fedora-welcome:128
msgid ""
"You can choose \"Install to Hard Drive\"\n"
"in the Activities Overview at any later time."
msgstr ""
"後にでも「アクティビティオーバービュー」内で 「ハードドライブにインストール」"
"を選択することができます。"

#: data/liveinst/gnome/fedora-welcome:131
msgid "Close"
msgstr "閉じる"

#~ msgid ""
#~ "Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
#~ "'0-9'"
#~ msgstr ""
#~ "ホスト名は 「a-z」 または 「A-Z」または 「0-9」 の範囲の有効な文字で開始し"
#~ "なければ なりません。"

#~ msgid "You need more space on the following file systems:\n"
#~ msgstr "以下のファイルシステムは容量を増やす必要があります。\n"

#~ msgid "Can't have a question in command line mode!"
#~ msgstr "コマンドラインモードでは質問は置けません!"

#, fuzzy
#~ msgid "That name is invalid. Try something else?"
#~ msgstr "そのマウントポイントは無効です。他を試しますか?"

#, fuzzy
#~ msgid "_NETWORK CONFIGURATION"
#~ msgstr "ネットワーク設定"

#~ msgid "Error downloading package metadata..."
#~ msgstr "パッケージメタデータをダウンロード中にエラー..."

#, fuzzy
#~ msgid "Nothing here yet."
#~ msgstr "何も選択されていません"

#, fuzzy
#~ msgid "Firmware _RAID"
#~ msgstr "ファームウェアがありません"

#, fuzzy
#~ msgid "<b>Authentication required by wireless network</b>"
#~ msgstr "認証が必要です"

#~ msgid "column"
#~ msgstr "列"

#, fuzzy
#~ msgid "<b>Partition scheme:</b>"
#~ msgstr "<b>パーティションタイプ:</b>"

#, fuzzy
#~ msgid "<b>_Proxy URL</b>"
#~ msgstr "<b>プロキシの URL</b>"

#, fuzzy
#~ msgid "<b>User_name</b>"
#~ msgstr "<b>ユーザー名</b>"

#, fuzzy
#~ msgid "<b>Pass_word</b>"
#~ msgstr "<b>パスワード</b>"

#, fuzzy
#~ msgid "Invalid repository name. Try something else?"
#~ msgstr "そのマウントポイントは無効です。他を試しますか?"

#, fuzzy
#~ msgid "<b>_Label:</b>"
#~ msgstr "<b>ラベル:</b>"

#, fuzzy
#~ msgid "<b>File _System:</b>"
#~ msgstr "<b>ファイルシステム:</b>"

#, fuzzy
#~ msgid "R_AID Level:"
#~ msgstr "RAID デバイス"

#~ msgid "Configure selected mountpoint."
#~ msgstr "選択したマウントポイントの設定を行います。"

#, fuzzy
#~ msgid "Click for help."
#~ msgstr "ヘルプをクリックします"

#~ msgid "<span size=\"small\" foreground=\"white\">AVAILABLE SPACE</span>"
#~ msgstr "<span size=\"small\" foreground=\"white\">利用可能な領域</span>"

#~ msgid "<span size=\"small\" foreground=\"white\">TOTAL SPACE</span>"
#~ msgstr "<span size=\"small\" foreground=\"white\">合計領域</span>"

#~ msgid "Completed"
#~ msgstr "完了"

#~ msgid ""
#~ "(%s)\n"
#~ "Command line mode requires all choices to be specified in a kickstart "
#~ "configuration file."
#~ msgstr ""
#~ "(%s)\n"
#~ "コマンドラインモードではすべての選択肢がキックスタート設定ファイルで指定さ"
#~ "れている必要があります。"

#~ msgid ""
#~ "The following error was found while parsing the kickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "キックスタート設定ファイルの解析中に次のエラーが見つかりました。\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "(%s)\n"
#~ "%s"
#~ msgstr ""
#~ "(%s)\n"
#~ "%s"

#~ msgid "In interactive step can't continue. (%s)"
#~ msgstr "対話ステップを続行できません。(%s)"

#~ msgid ""
#~ "An unhandled exception has occurred.  This is most likely a bug.  Please "
#~ "save a copy of the detailed exception and file a bug report"
#~ msgstr ""
#~ "未処理の例外が発生しました。バグの可能性が考えられます。すべてのメッセージ"
#~ "をコピーし、バグとして報告してください。"

#~ msgid " with the provider of this software."
#~ msgstr "このソフトウェアの提供者と共に"

#~ msgid " against anaconda at %s"
#~ msgstr "anaconda の %s に対して"

#~ msgid "Running post install scripts"
#~ msgstr "インストールの後処理スクリプトを実行中"

#~ msgid "Install on System"
#~ msgstr "システム上にインストール"

#~ msgid "Exit installer"
#~ msgstr "インストーラーを終了する"

#~ msgid "_Exit installer"
#~ msgstr "インストーラーを終了する(_E)"

#~ msgid ""
#~ "After shutdown, please perform a manual IPL from %s to continue "
#~ "installation."
#~ msgstr ""
#~ "シャットダウンの後に、%s から手動で IPL を実行してインストールを継続してく"
#~ "ださい"

#~ msgid "IP address is missing."
#~ msgstr "IP アドレスがありません。"

#~ msgid ""
#~ "IPv4 addresses must contain four numbers between 0 and 255, separated by "
#~ "periods."
#~ msgstr ""
#~ "IPv4 アドレスは、ピリオドで区切られた 0 から 255 までの 4 つの数字を含まな"
#~ "ければなりません。"

#~ msgid "'%s' is not a valid IPv6 address."
#~ msgstr "%s は有効な IPv6 アドレスではありません。"

#~ msgid "'%s' is an invalid IP address."
#~ msgstr "'%s' は、有効な IP アドレスではありません。"

#~ msgid "bootloader device"
#~ msgstr "ブートローダーデバイス"

#~ msgid "First sector of boot partition"
#~ msgstr "ブートパーティションの最初のセクタ"

#~ msgid "Master Boot Record"
#~ msgstr "Master Boot Record"

#~ msgid "EFI System Partition"
#~ msgstr "EFI システムパーティション"

#~ msgid "Apple EFI Boot Partition"
#~ msgstr "Apple EFI Boot パーティション"

#~ msgid "PReP Boot Partition"
#~ msgstr "PReP ブートパーティション"

#~ msgid "Apple Bootstrap Partition"
#~ msgstr "Apple ブートストラップパーティション"

#~ msgid "DASD"
#~ msgstr "DASD"

#~ msgid "zFCP"
#~ msgstr "zFCP"

#~ msgid "U-Boot Partition"
#~ msgstr "U-Boot パーティション"

#~ msgid "%s MB"
#~ msgstr "%s MB"

#~ msgid "%s KB"
#~ msgstr "%s KB"

#~ msgid "%s Byte"
#~ msgid_plural "%s Bytes"
#~ msgstr[0] "%s バイト"

#~ msgid "<b>Upgrading %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>%(pkgStr)s をアップグレード中</b> (%(size)s)\n"

#~ msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
#~ msgstr "<b>%(pkgStr)s のインストール</b> (%(size)s)\n"

#~ msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
#~ msgstr[0] "完了したパッケージ: %(numpkgs)d 個中 %(donepkgs)d 個が完了"

#~ msgid "<b>Cleaning up %s</b>"
#~ msgstr "<b>%s をクリーンナップ中</b>"

#~ msgid "cpio"
#~ msgstr "cpio"

#~ msgid "unpack"
#~ msgstr "展開"

#~ msgid "script"
#~ msgstr "スクリプト"

#~ msgid "Error Installing Package"
#~ msgstr "パッケージのインストールでエラー"

#~ msgid ""
#~ "A %s error occurred when installing the %s package.  This could indicate "
#~ "errors when reading the installation media.  Installation cannot continue."
#~ msgstr ""
#~ "%s エラーが %s パッケージのインストール中に発生しました。インストールメ"
#~ "ディアの読み込み中に問題が発生した可能性があります。インストール作業を続行"
#~ "できません。"

#~ msgid "Wrong Disc"
#~ msgstr "不適切なディスクです"

#~ msgid "That's not the correct %s disc."
#~ msgstr "正しい %s ディスクではありません。"

#~ msgid "Repository %r is missing name in configuration, using id"
#~ msgstr "リポジトリー %r は設定内で名前がありません。id を使用します。"

#~ msgid "No Network Available"
#~ msgstr "ネットワークは使用できません。"

#~ msgid ""
#~ "Some of your software repositories require networking, but there was an "
#~ "error enabling the network on your system."
#~ msgstr ""
#~ "ソフトウェアリポジトリーの一部はネットワークを必要とします。しかし、使用中"
#~ "のシステムでネットワークの有効化にエラーが発生しました。"

#~ msgid "Re_boot"
#~ msgstr "再起動(_B)"

#~ msgid "_Eject"
#~ msgstr "ディスクを取り出す(_E)"

#~ msgid "_Retry"
#~ msgstr "再試行(_R)"

#~ msgid ""
#~ "The file %s cannot be opened.  This is due to a missing file, a corrupt "
#~ "package or corrupt media.  Please verify your installation source.\n"
#~ "\n"
#~ "If you exit, your system will be left in an inconsistent state that will "
#~ "likely require reinstallation.\n"
#~ "\n"
#~ msgstr ""
#~ "ファイル %s を開けません。ファイルが見つからないか、パッケージやメディアが"
#~ "破損している可能性があります。インストール元を検証してください。\n"
#~ "\n"
#~ "終了するとシステムは矛盾した状態となり、再インストールを必要とする可能性が"
#~ "高くなります。\n"
#~ "\n"

#~ msgid "Retrying"
#~ msgstr "再試行中"

#~ msgid "Retrying download."
#~ msgstr "ダウンロードの再試行中。"

#~ msgid ""
#~ "There was an error running your transaction for the following reason: %s\n"
#~ msgstr ""
#~ "トランザクションの実行中に次のような理由でエラーが発生しました。%s\n"

#~ msgid "_Back"
#~ msgstr "戻る(_B)"

#~ msgid ""
#~ "There was an error running your transaction for the following reason: "
#~ "%s.\n"
#~ msgstr "トランザクションを実行中に次のエラーが発生しました: %s\n"

#~ msgid "Error Running Transaction"
#~ msgstr "トランザクション実行中にエラーが発生しました。"

#~ msgid "file conflicts"
#~ msgstr "ファイル競合"

#~ msgid "older package(s)"
#~ msgstr "旧パッケージ"

#~ msgid "insufficient disk space"
#~ msgstr "ディスク容量の不足"

#~ msgid "insufficient disk inodes"
#~ msgstr "ディスク inode の不足"

#~ msgid "package conflicts"
#~ msgstr "パッケージ競合"

#~ msgid "package already installed"
#~ msgstr "パッケージはすでにインストールされています"

#~ msgid "required package"
#~ msgstr "必要なパッケージ"

#~ msgid "package for incorrect arch"
#~ msgstr "不適切なアーキテクチャ用のパッケージ"

#~ msgid "package for incorrect os"
#~ msgstr "不適切な OS 用のパッケージ"

#~ msgid ""
#~ "There were file conflicts when checking the packages to be installed:\n"
#~ "%s\n"
#~ msgstr ""
#~ "インストールするパッケージに、競合するファイルがみつかりました:\n"
#~ "%s\n"

#~ msgid ""
#~ "There was an error running your transaction for the following reason(s): "
#~ "%s.\n"
#~ msgstr ""
#~ "トランザクションの実行中に次のような理由でエラーが発生しました: %s\n"

#~ msgid ""
#~ "Unable to read group information from repositories.  This is a problem "
#~ "with the generation of your install tree."
#~ msgstr ""
#~ "リポジトリーからグループ情報を読み取ることはできません。これは、インストー"
#~ "ルツリーの生成の問題です。"

#~ msgid "Edit"
#~ msgstr "編集"

#~ msgid ""
#~ "Unable to read package metadata. This may be due to a missing repodata "
#~ "directory.  Please ensure that your install tree has been correctly "
#~ "generated.\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "パッケージのメタデータを読めません。repodata ディレクトリーが見つからない"
#~ "のが原因かもしれません。インストールツリーが正しく構築されているか確認して"
#~ "ください。\n"
#~ "\n"
#~ "%s"

#~ msgid ""
#~ "Some of the packages you have selected for install are missing "
#~ "dependencies.  You can exit the installation, go back and change your "
#~ "package selections, or continue installing these packages without their "
#~ "dependencies.  If you continue, these packages may not work correctly due "
#~ "to missing components."
#~ msgstr ""
#~ "インストールするために選択したパッケージの一部は依存関係が欠如していま"
#~ "す。 インストールを終了してパッケージ選択に戻るか、または依存関係のない"
#~ "パッケージの インストールを継続してください。継続する場合、欠如したコン"
#~ "ポーネントのために これらのパッケージは正常に機能しない可能性があります。"

#~ msgid ""
#~ "Your selected packages require %d MB of free space for installation, but "
#~ "you do not have enough available.  You can change your selections or exit "
#~ "the installer."
#~ msgstr ""
#~ "選択したパッケージはインストールに %d MB の空き領域を必要としますが、充分"
#~ "な空きがありません。選択を変更するか、またはインストーラーを終了してくださ"
#~ "い。"

#~ msgid "Post Upgrade"
#~ msgstr "アップグレード後"

#~ msgid "Performing post-upgrade configuration"
#~ msgstr "アップグレード後の設定を実行中。"

#~ msgid "Post Installation"
#~ msgstr "インストール後"

#~ msgid "Performing post-installation configuration"
#~ msgstr "インストール後の設定を実行中"

#~ msgid "Installation Starting"
#~ msgstr "インストールを開始しています"

#~ msgid "Starting installation process"
#~ msgstr "インストール処理の開始"

#~ msgid "Dependency Check"
#~ msgstr "依存関係チェック"

#~ msgid "Checking dependencies in packages selected for installation"
#~ msgstr "インストール対象パッケージの依存関係をチェック中。"

#~ msgid "Retrieving installation information."
#~ msgstr "インストール情報を読み出し中。"

#~ msgid "Retrieving installation information for %s."
#~ msgstr "%s に関するインストール情報を読み出し中。"

#~ msgid ""
#~ "The default installation of %s includes a set of software applicable for "
#~ "general internet usage. You can optionally select a different set of "
#~ "software now."
#~ msgstr ""
#~ "%s のデフォルトインストールには一般的なネットワーク環境で利用できるソフト"
#~ "ウェアのセットが含まれています。オプションとして異なる組み合わせのソフト"
#~ "ウェアを選択することができます。"

#~ msgid "Graphical Desktop"
#~ msgstr "デスクトップ"

#~ msgid "Software Development"
#~ msgstr "ソフトウェア開発"

#~ msgid "Minimal"
#~ msgstr "最小限"

#~ msgid ""
#~ "The default installation of %s is a minimum install. You can optionally "
#~ "select a different set of software now."
#~ msgstr ""
#~ "%s のデフォルトインストールは最小限インストールです。オプションとして追加"
#~ "のソフトウェアを選択することができます。"

#~ msgid "Upgrading"
#~ msgstr "アップグレード中"

#~ msgid "This partition is holding the data for the hard drive install."
#~ msgstr ""
#~ "このパーティションはハードディスクインストール用のデータを保有しています。"

#~ msgid "You cannot delete a partition of a LDL formatted DASD."
#~ msgstr "DASD 形式でフォーマットされた LDL パーティションは削除できません。"

#~ msgid "This device is part of the RAID device %s."
#~ msgstr "このデバイスは RAID デバイス %s の一部です。"

#~ msgid "This device is part of a RAID device."
#~ msgstr "このデバイスは RAID デバイスの一部です。"

#~ msgid "This device is part of an inconsistent LVM Volume Group."
#~ msgstr "このデバイスは一貫性のない LVM ボリュームグループの一部です。"

#~ msgid "This device is part of the LVM volume group '%s'."
#~ msgstr "このデバイスは LVM ボリュームグループ %s の一部です。"

#~ msgid "This device is part of a LVM volume group."
#~ msgstr "このデバイスは LVM ボリュームグループの一部です。"

#~ msgid ""
#~ "This device is an extended partition which contains logical partitions "
#~ "that cannot be deleted:\n"
#~ "\n"
#~ msgstr ""
#~ "このデバイスは拡張パーティションです。この中の論理パーティションは先に 削"
#~ "除できません:\n"
#~ "\n"

#~ msgid ""
#~ "Installing on a USB device.  This may or may not produce a working system."
#~ msgstr ""
#~ "USB デバイス上にインストールしています。稼動するシステムが作成されることも"
#~ "ありますが、されないこともあります。"

#~ msgid ""
#~ "Installing on a FireWire device.  This may or may not produce a working "
#~ "system."
#~ msgstr ""
#~ "FireWire デバイス上にインストールしています。稼動するシステムが作成される"
#~ "こともありますが、されないこともあります。"

#~ msgid "you have not created a bootloader stage1 target device"
#~ msgstr ""
#~ "対象のデバイスでブートローダーの stage1 を作成することができませんでした。"

#~ msgid "Linux on %s"
#~ msgstr "%s 上の Linux"

#~ msgid "Unknown Linux"
#~ msgstr "未知の Linux"

#~ msgid "%(product)s Linux %(version)s for %(arch)s"
#~ msgstr "%(product)s Linux %(version)s － %(arch)s"

#~ msgid "Preparing %d DASD device for use with Linux..."
#~ msgid_plural "Preparing %d DASD devices for use with Linux..."
#~ msgstr[0] "Linuxで使うために %d 個のDASDのデバイスを準備中..."

#~ msgid "Resizing filesystem on %(device)s"
#~ msgstr "%(device)s のファイルシステムの容量変更中"

#~ msgid "Migrating filesystem on %(device)s"
#~ msgstr "%(device)s のファイルシステムを移行中"

#~ msgid "vginfo failed for %s"
#~ msgstr "%s 用の vginfo に失敗"

#~ msgid "lvs failed for %s"
#~ msgstr "%s 用の lvs に失敗"

#~ msgid "A RAID%(raidLevel)d set requires at least %(minMembers)d member"
#~ msgid_plural ""
#~ "A RAID%(raidLevel)d set requires at least %(minMembers)d members"
#~ msgstr[0] ""
#~ "RAID%(raidLevel)d には少なくても %(minMembers)d 個のディスクが必要です。"

#~ msgid "Back"
#~ msgstr "戻る"

#~ msgid "Confirm"
#~ msgstr "確認"

#~ msgid ""
#~ "Are you sure you want to skip entering a passphrase for device %s?\n"
#~ "\n"
#~ "If you skip this step the device's contents will not be available during "
#~ "installation."
#~ msgstr ""
#~ "本当にデバイス %s のパスフレーズの入力を省略しますか?\n"
#~ "\n"
#~ "このステップをスキップすると、インストール中のデバイス内のコンテンツへのア"
#~ "クセスはできません。"

#~ msgid "FCoE not available"
#~ msgstr "FCoE は使用できません"

#~ msgid "Unknown return code: %d."
#~ msgstr "不明なリターンコードです: %d"

#~ msgid "%(type)s filesystem check failure on %(device)s: "
#~ msgstr "%(device)s 上で %(type)s ファイルシステムのチェックに失敗: "

#~ msgid "Unrecoverable Error"
#~ msgstr "修正不可能なエラー"

#~ msgid "File system errors left uncorrected."
#~ msgstr "ファイルシステムエラーが未修正のままです。"

#~ msgid "Usage or syntax error."
#~ msgstr "ユーザーまたは構文のエラー"

#~ msgid "e2fsck cancelled by user request."
#~ msgstr "ユーザーの指示で e2fsck がキャンセルされました。"

#~ msgid "Shared library error."
#~ msgstr "共有ライブラリのエラー"

#~ msgid ""
#~ "Recoverable errors have been detected or dosfsck has discovered an "
#~ "internal inconsistency."
#~ msgstr ""
#~ "復元可能なエラーが検出されたか、または dosfsck が内部の整合性問題を発見し"
#~ "ました。"

#~ msgid "Usage error."
#~ msgstr "使用法エラー"

#~ msgid "Encrypted"
#~ msgstr "暗号化"

#~ msgid "Unable to change iSCSI initiator name once set"
#~ msgstr "iSCSI イニシエータ名はセットされた後は変更できません"

#~ msgid "iSCSI not available"
#~ msgstr "iSCSI は使用できません"

#~ msgid "No initiator name set"
#~ msgstr "イニシエーター名がありません"

#~ msgid "Logging in to iSCSI node"
#~ msgstr "iSCSI ノードにログイン中"

#~ msgid "Logging in to iSCSI node %s"
#~ msgstr "iSCSI ノード %s にログイン中"

#~ msgid "No iSCSI nodes discovered"
#~ msgstr "iSCSI ノードが見つかりませんでした"

#~ msgid "No new iSCSI nodes discovered"
#~ msgstr "新しい iSCSI ノードは見つかりませんでした。"

#~ msgid "Could not log in to any of the discovered nodes"
#~ msgstr "見つかったノードすべてにログインできませんでした"

#~ msgid "No usable disks selected"
#~ msgstr "使用可能なディスクが未選択"

#~ msgid "Not enough free space on disks for automatic partitioning"
#~ msgstr ""
#~ "自動パーティション構成のために必要なディスクの空き領域が不足しています。"

#~ msgid "unable to allocate aligned partition"
#~ msgstr "アラインされたパーティションを割り当てることができません。"

#~ msgid "can't allocate partitions without platform data"
#~ msgstr ""
#~ "プラットフォームデータなしでパーティションを割り当てることはできません。"

#~ msgid "disk %s inaccessible"
#~ msgstr "ディスク %s はアクセスできません"

#~ msgid ""
#~ "partition is too small for %(format)s formatting (allowable size is "
#~ "%(minSize)d MB to %(maxSize)d MB)"
#~ msgstr ""
#~ "%(format)s でフォーマットするにはパーティションが小さ過ぎます。(割り当てサ"
#~ "イズは %(minSize)d MB から %(maxSize)d MB までが指定可能)"

#~ msgid ""
#~ "partition is too large for %(format)s formatting (allowable size is "
#~ "%(minSize)d MB to %(maxSize)d MB)"
#~ msgstr ""
#~ "%(format)s でフォーマットするにはパーティションが大き過ぎます。(割り当てサ"
#~ "イズは %(minSize)d MB から %(maxSize)d MB までが指定可能)"

#~ msgid "not enough free space on disks"
#~ msgstr "ディスクの空き領域が不足しています"

#~ msgid "not enough free space after creating extended partition"
#~ msgstr ""
#~ "拡張パーティションを作成するためのディスクの空き領域が不足しています。"

#~ msgid "DiskChunk requests must be of type PartitionRequest"
#~ msgstr ""
#~ "DiskChunk リクエストは PartitionRequest のタイプでなければなりません。"

#~ msgid "partitions allocated outside disklabel limits"
#~ msgstr "ディスクラベルの上限を超える割り当てパーティション"

#~ msgid "VGChunk requests must be of type LVRequest"
#~ msgstr "VGChunk リクエストは LVRequest のタイプでなければなりません。"

#~ msgid "not enough space for LVM requests"
#~ msgstr "LVM リクエストのために空き領域が不足しています。"

#~ msgid "You have not specified a device number or the number is invalid"
#~ msgstr "デバイス番号を指定していないか番号が無効です。"

#~ msgid "You have not specified a worldwide port name or the name is invalid."
#~ msgstr "WWPN を指定していないか、そのポート名が無効です。"

#~ msgid "You have not specified a FCP LUN or the number is invalid."
#~ msgstr "FCP FUN を指定していないか、番号が無効です。"

#~ msgid "zFCP device %s not found, not even in device ignore list."
#~ msgstr "zFCP デバイス %s が見つからず、デバイス無視リストにもありません。"

#~ msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s を オンライン (%(e)s) にセットできませんでした。"

#~ msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "WWPN %(wwpn)s を zFCP デバイス %(devnum)s (%(e)s) に追加できません。"

#~ msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
#~ msgstr "WWPN %(wwpn)s が zFCP デバイス %(devnum)s で見つかりません。"

#~ msgid ""
#~ "Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
#~ "(%(e)s)."
#~ msgstr ""
#~ "LUN %(fcplun)s を zFCP デバイス %(devnum)s (%(e)s)上で WWPN %(wwpn)s に追"
#~ "加できません、。"

#~ msgid ""
#~ "LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already "
#~ "configured."
#~ msgstr ""
#~ "LUN %(fcplun)s は、zFCP デバイス %(devnum)s 上の WWPN %(wwpn)s ですでに設"
#~ "定されています。"

#~ msgid ""
#~ "Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on "
#~ "zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s (%(e)s) 上の WWPN %(wwpn)s で LUN %(fcplun)s の "
#~ "失敗属性を読みこむことが できませんでした。"

#~ msgid ""
#~ "Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
#~ "again."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s 上の WWPN %(wwpn)s で失敗した LUN %(fcplun)s が "
#~ "再度削除されました。"

#~ msgid ""
#~ "Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
#~ "%(fcplun)s (%(e)s)."
#~ msgstr ""
#~ "zFCP %(devnum)s %(wwpn)s %(fcplun)s (%(e)s) の SCSI デバイスを正しく消すこ"
#~ "とができませんでした。"

#~ msgid ""
#~ "Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device "
#~ "%(devnum)s (%(e)s)."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s (%(e)s) 上の WWPN %(wwpn)s で LUN %(fcplun)s を削"
#~ "除できませんでした。"

#~ msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s (%(e)s) 上で WWPN %(wwpn)s を削除できませんでし"
#~ "た。"

#~ msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
#~ msgstr ""
#~ "zFCP デバイス %(devnum)s をオフライン (%(e)s) に設定できませんでした。"

#~ msgid "Install hub"
#~ msgstr "インストールハブ"

#~ msgid "You have provided a weak password: %s. "
#~ msgstr "強度が弱いパスワードを入力しています: %s"

#~ msgid ""
#~ "\n"
#~ "Would you like to use it anyway?"
#~ msgstr ""
#~ "\n"
#~ "とにかくこれを使用しますか?"

#~ msgid "STORAGE"
#~ msgstr "ストレージ"

#~ msgid ""
#~ "When all items marked with this icon are complete, installation will "
#~ "automatically continue."
#~ msgstr ""
#~ "このアイコンの付いたすべての項目が完了すると、自動的にインストールが継続し"
#~ "ます。"

#~ msgid "Note: pool servers may not be available all the time"
#~ msgstr "注記:  プールサーバーは常時利用可能ではないかも知れません。"

#~ msgid "Connected: %s"
#~ msgstr "接続しています: %s"

#~ msgid "Error setting root password"
#~ msgstr "root パスワードの設定エラーです"

#~ msgid "_Remove Packages"
#~ msgstr "パッケージを削除 (_R)"

#~ msgid "Getting info about requested repository"
#~ msgstr "要求されたレポジトリの情報を取得中…"

#~ msgid "enter URL of your desired repository"
#~ msgstr "ご希望のレポジトリの URL を入力してください"

#~ msgid ""
#~ "Your current <b>%s</b> software selection requires <b>%s</b> of available "
#~ "space."
#~ msgstr ""
#~ "現在の <b>%s</b> のソフトウェア選択には、利用可能な領域の <b>%s</b> が必要"
#~ "です。"

#~ msgid "Whole disks are not editable."
#~ msgstr "ディスクは編集できません"

#~ msgid "%s of %s"
#~ msgstr "%s のうち %s"

#~ msgid "00"
#~ msgstr "00"

#~ msgid " "
#~ msgstr " "

#~ msgid "Choose your add-ons"
#~ msgstr "使用アドオンを選択します"

#~ msgid "_Add custom add-on..."
#~ msgstr "カスタムアドオンを追加(_A)"

#~ msgid "Let me customize the _partitioning of the disks instead."
#~ msgstr ""
#~ "代わりにディスクのパーティション設定をカスタマイズさせて下さい。(_P)"

#~ msgid "Reclaim _space"
#~ msgstr "領域を取り戻す(_S)"

#~ msgid "disks size"
#~ msgstr "ディスクのサイズ"

#~ msgid "Space in selected disks reclaimable by deleting existing partitions."
#~ msgstr "既存パーティションの削除により選択したディスク内の獲得できる領域"

#~ msgid "I don't need help; let me customize disk _partitioning."
#~ msgstr ""
#~ "ヘルプは必要ありません。 ディスクのパーティション設定をカスタマイズしま"
#~ "す。"

#~ msgid "Test the selected layout below:"
#~ msgstr "選択したレイアウトでテストしてください:"

#~ msgid "ADD A CUSTOM ADD-ON"
#~ msgstr "カスタムアドオンを追加"

#~ msgid "Where is the yum repository for your add-on?"
#~ msgstr "アドオン用の yum リポジトリを選択してください。"

#~ msgid "Repository"
#~ msgstr "レポジトリ"

#~ msgid "Standard YUM repository:"
#~ msgstr "標準の YUM レポジトリ"

#~ msgid "Set _keyboard to default layout for selected language."
#~ msgstr "選択した言語でデフォルトのキーボードに設定(_K)"

#~ msgid "eg: \"/\", \"/home\", \"swap\" (minus the quotation marks)"
#~ msgstr "例: \"/\"、 \"/home\"、 \"swap\" (引用符は除く)"

#~ msgid "_Back to destination selection"
#~ msgstr "目的地選択に戻る(_B)"

#~ msgid "<b>Desired Capacity (MB):</b>"
#~ msgstr "<b>要望する容量 (MB):</b>"

#~ msgid "Redundancy (mirror)"
#~ msgstr "冗長 (ミラー)"

#~ msgid "(-0Gb)"
#~ msgstr "(-0Gb)"

#~ msgid "Optimized performance (stripe)"
#~ msgstr "最適化したパフォーマンス (ストライプ)"

#~ msgid "Error detection (parity)"
#~ msgstr "エラー検出 (パリティ)"

#~ msgid "Distributed"
#~ msgstr "分散"

#~ msgid "Redundant"
#~ msgstr "冗長"

#~ msgid "_Customize..."
#~ msgstr "カスタマイズする(_C)..."

#~ msgid ""
#~ "This %s device is missing %d of %d %s. You can remove it or select a "
#~ "different device."
#~ msgstr ""
#~ "%s デバイスの %d / %d の %s が見つかりません。削除するか他のデバイスを選択"
#~ "してください。"

#~ msgid "addButton"
#~ msgstr "追加ボタン"

#~ msgid "removeButton"
#~ msgstr "removeButton"

#~ msgid "configureButton"
#~ msgstr "設定ボタン"

#~ msgid "CONFIGURE VOLUME GROUP"
#~ msgstr "構成するボリュームグループ"

#~ msgid "Id"
#~ msgstr "ID"

#~ msgid "keymap warning label"
#~ msgstr "キー対応表警告ラベル"

#~ msgid "<b>%s disks; %s reclaimable space</b> (in filesystems)"
#~ msgstr "<b>%s 個のディスク; %s つの獲得可能領域</b> (ファイルシステム内)"

#~ msgid "Total selected space to reclaim:  <b>%s</b>"
#~ msgstr "選択した取得する領域の合計:　<b>%s</b>"

#~ msgid "Installation requires a total of <b>%s</b> for system data."
#~ msgstr "インストールには、システムデータ用に合計 <b>%s</b> が必要です。"

#~ msgid "Root"
#~ msgstr "Root"
